diff -U 0 -N multidex-instrumentation-1.0.0_eb3081c4/android.support.multidex.BuildConfig multidex-instrumentation-1.0.1_d8c206f6/android.support.multidex.BuildConfig
--- multidex-instrumentation-1.0.0_eb3081c4/android.support.multidex.BuildConfig	2015-09-05 18:52:09.000000000 +0900
+++ multidex-instrumentation-1.0.1_d8c206f6/android.support.multidex.BuildConfig	1970-01-01 09:00:00.000000000 +0900
@@ -1,9 +0,0 @@
-public final class android.support.multidex.BuildConfig {
-  public static final boolean DEBUG;
-  public static final java.lang.String PACKAGE_NAME;
-  public static final java.lang.String BUILD_TYPE;
-  public static final java.lang.String FLAVOR;
-  public static final int VERSION_CODE;
-  public static final java.lang.String VERSION_NAME;
-  public android.support.multidex.BuildConfig();
-}
diff -U 0 -N multidex-instrumentation-1.0.0_eb3081c4/android.support.multidex.instrumentation.BuildConfig multidex-instrumentation-1.0.1_d8c206f6/android.support.multidex.instrumentation.BuildConfig
--- multidex-instrumentation-1.0.0_eb3081c4/android.support.multidex.instrumentation.BuildConfig	1970-01-01 09:00:00.000000000 +0900
+++ multidex-instrumentation-1.0.1_d8c206f6/android.support.multidex.instrumentation.BuildConfig	2015-09-05 18:52:09.000000000 +0900
@@ -0,0 +1,9 @@
+public final class android.support.multidex.instrumentation.BuildConfig {
+  public static final boolean DEBUG;
+  public static final java.lang.String PACKAGE_NAME;
+  public static final java.lang.String BUILD_TYPE;
+  public static final java.lang.String FLAVOR;
+  public static final int VERSION_CODE;
+  public static final java.lang.String VERSION_NAME;
+  public android.support.multidex.instrumentation.BuildConfig();
+}
