diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.animation.AnimatorListenerCompat support-compat-25.1.0_eca2886e/android.support.v4.animation.AnimatorListenerCompat
--- support-compat-25.0.1_58db3096/android.support.v4.animation.AnimatorListenerCompat	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.animation.AnimatorListenerCompat	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public interface android.support.v4.animation.AnimatorListenerCompat {
+  public abstract void onAnimationStart(android.support.v4.animation.ValueAnimatorCompat);
+  public abstract void onAnimationEnd(android.support.v4.animation.ValueAnimatorCompat);
+  public abstract void onAnimationCancel(android.support.v4.animation.ValueAnimatorCompat);
+  public abstract void onAnimationRepeat(android.support.v4.animation.ValueAnimatorCompat);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.animation.AnimatorUpdateListenerCompat support-compat-25.1.0_eca2886e/android.support.v4.animation.AnimatorUpdateListenerCompat
--- support-compat-25.0.1_58db3096/android.support.v4.animation.AnimatorUpdateListenerCompat	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.animation.AnimatorUpdateListenerCompat	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.animation.AnimatorUpdateListenerCompat {
+  public abstract void onAnimationUpdate(android.support.v4.animation.ValueAnimatorCompat);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.animation.ValueAnimatorCompat support-compat-25.1.0_eca2886e/android.support.v4.animation.ValueAnimatorCompat
--- support-compat-25.0.1_58db3096/android.support.v4.animation.ValueAnimatorCompat	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.animation.ValueAnimatorCompat	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,9 @@
+public interface android.support.v4.animation.ValueAnimatorCompat {
+  public abstract void setTarget(android.view.View);
+  public abstract void addListener(android.support.v4.animation.AnimatorListenerCompat);
+  public abstract void setDuration(long);
+  public abstract void start();
+  public abstract void cancel();
+  public abstract void addUpdateListener(android.support.v4.animation.AnimatorUpdateListenerCompat);
+  public abstract float getAnimatedFraction();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi21$SharedElementCallback21 support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi21$SharedElementCallback21
--- support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi21$SharedElementCallback21	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi21$SharedElementCallback21	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,9 @@
+public abstract class android.support.v4.app.ActivityCompatApi21$SharedElementCallback21 {
+  public android.support.v4.app.ActivityCompatApi21$SharedElementCallback21();
+  public abstract void onSharedElementStart(java.util.List<java.lang.String>, java.util.List<android.view.View>, java.util.List<android.view.View>);
+  public abstract void onSharedElementEnd(java.util.List<java.lang.String>, java.util.List<android.view.View>, java.util.List<android.view.View>);
+  public abstract void onRejectSharedElements(java.util.List<android.view.View>);
+  public abstract void onMapSharedElements(java.util.List<java.lang.String>, java.util.Map<java.lang.String, android.view.View>);
+  public abstract android.os.Parcelable onCaptureSharedElementSnapshot(android.view.View, android.graphics.Matrix, android.graphics.RectF);
+  public abstract android.view.View onCreateSnapshotView(android.content.Context, android.os.Parcelable);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi23$OnSharedElementsReadyListenerBridge support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi23$OnSharedElementsReadyListenerBridge
--- support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi23$OnSharedElementsReadyListenerBridge	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi23$OnSharedElementsReadyListenerBridge	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.app.ActivityCompatApi23$OnSharedElementsReadyListenerBridge {
+  public abstract void onSharedElementsReady();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi23$RequestPermissionsRequestCodeValidator support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi23$RequestPermissionsRequestCodeValidator
--- support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi23$RequestPermissionsRequestCodeValidator	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi23$RequestPermissionsRequestCodeValidator	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.app.ActivityCompatApi23$RequestPermissionsRequestCodeValidator {
+  public abstract void validateRequestPermissionsRequestCode(int);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi23$SharedElementCallback23 support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi23$SharedElementCallback23
--- support-compat-25.0.1_58db3096/android.support.v4.app.ActivityCompatApi23$SharedElementCallback23	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.ActivityCompatApi23$SharedElementCallback23	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public abstract class android.support.v4.app.ActivityCompatApi23$SharedElementCallback23 extends android.support.v4.app.ActivityCompatApi21$SharedElementCallback21 {
+  public android.support.v4.app.ActivityCompatApi23$SharedElementCallback23();
+  public abstract void onSharedElementsArrived(java.util.List<java.lang.String>, java.util.List<android.view.View>, android.support.v4.app.ActivityCompatApi23$OnSharedElementsReadyListenerBridge);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationBuilderWithBuilderAccessor support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationBuilderWithBuilderAccessor
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationBuilderWithBuilderAccessor	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationBuilderWithBuilderAccessor	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public interface android.support.v4.app.NotificationBuilderWithBuilderAccessor {
+  public abstract android.app.Notification$Builder getBuilder();
+  public abstract android.app.Notification build();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatApi20$Builder support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatApi20$Builder
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatApi20$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatApi20$Builder	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.app.NotificationCompatApi20$Builder implements android.support.v4.app.NotificationBuilderWithBuilderAccessor,android.support.v4.app.NotificationBuilderWithActions {
+  public android.support.v4.app.NotificationCompatApi20$Builder(android.content.Context, android.app.Notification, java.lang.CharSequence, java.lang.CharSequence, java.lang.CharSequence, android.widget.RemoteViews, int, android.app.PendingIntent, android.app.PendingIntent, android.graphics.Bitmap, int, int, boolean, boolean, boolean, int, java.lang.CharSequence, boolean, java.util.ArrayList<java.lang.String>, android.os.Bundle, java.lang.String, boolean, java.lang.String, android.widget.RemoteViews, android.widget.RemoteViews);
+  public void addAction(android.support.v4.app.NotificationCompatBase$Action);
+  public android.app.Notification$Builder getBuilder();
+  public android.app.Notification build();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatApi21$Builder support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatApi21$Builder
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatApi21$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatApi21$Builder	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.app.NotificationCompatApi21$Builder implements android.support.v4.app.NotificationBuilderWithBuilderAccessor,android.support.v4.app.NotificationBuilderWithActions {
+  public android.support.v4.app.NotificationCompatApi21$Builder(android.content.Context, android.app.Notification, java.lang.CharSequence, java.lang.CharSequence, java.lang.CharSequence, android.widget.RemoteViews, int, android.app.PendingIntent, android.app.PendingIntent, android.graphics.Bitmap, int, int, boolean, boolean, boolean, int, java.lang.CharSequence, boolean, java.lang.String, java.util.ArrayList<java.lang.String>, android.os.Bundle, int, int, android.app.Notification, java.lang.String, boolean, java.lang.String, android.widget.RemoteViews, android.widget.RemoteViews, android.widget.RemoteViews);
+  public void addAction(android.support.v4.app.NotificationCompatBase$Action);
+  public android.app.Notification$Builder getBuilder();
+  public android.app.Notification build();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatApi24$Builder support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatApi24$Builder
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatApi24$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatApi24$Builder	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.app.NotificationCompatApi24$Builder implements android.support.v4.app.NotificationBuilderWithBuilderAccessor,android.support.v4.app.NotificationBuilderWithActions {
+  public android.support.v4.app.NotificationCompatApi24$Builder(android.content.Context, android.app.Notification, java.lang.CharSequence, java.lang.CharSequence, java.lang.CharSequence, android.widget.RemoteViews, int, android.app.PendingIntent, android.app.PendingIntent, android.graphics.Bitmap, int, int, boolean, boolean, boolean, int, java.lang.CharSequence, boolean, java.lang.String, java.util.ArrayList<java.lang.String>, android.os.Bundle, int, int, android.app.Notification, java.lang.String, boolean, java.lang.String, java.lang.CharSequence[], android.widget.RemoteViews, android.widget.RemoteViews, android.widget.RemoteViews);
+  public void addAction(android.support.v4.app.NotificationCompatBase$Action);
+  public android.app.Notification$Builder getBuilder();
+  public android.app.Notification build();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v4.app.NotificationCompatBase {
+  public android.support.v4.app.NotificationCompatBase();
+  public static android.app.Notification add(android.app.Notification, android.content.Context, java.lang.CharSequence, java.lang.CharSequence, android.app.PendingIntent, android.app.PendingIntent);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$Action support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$Action
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$Action	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$Action	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,9 @@
+public abstract class android.support.v4.app.NotificationCompatBase$Action {
+  public android.support.v4.app.NotificationCompatBase$Action();
+  public abstract int getIcon();
+  public abstract java.lang.CharSequence getTitle();
+  public abstract android.app.PendingIntent getActionIntent();
+  public abstract android.os.Bundle getExtras();
+  public abstract android.support.v4.app.RemoteInputCompatBase$RemoteInput[] getRemoteInputs();
+  public abstract boolean getAllowGeneratedReplies();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$Action$Factory support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$Action$Factory
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$Action$Factory	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$Action$Factory	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public interface android.support.v4.app.NotificationCompatBase$Action$Factory {
+  public abstract android.support.v4.app.NotificationCompatBase$Action build(int, java.lang.CharSequence, android.app.PendingIntent, android.os.Bundle, android.support.v4.app.RemoteInputCompatBase$RemoteInput[], boolean);
+  public abstract android.support.v4.app.NotificationCompatBase$Action[] newArray(int);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$UnreadConversation support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$UnreadConversation
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$UnreadConversation	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$UnreadConversation	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public abstract class android.support.v4.app.NotificationCompatBase$UnreadConversation {
+  public android.support.v4.app.NotificationCompatBase$UnreadConversation();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$UnreadConversation$Factory support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$UnreadConversation$Factory
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatBase$UnreadConversation$Factory	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatBase$UnreadConversation$Factory	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.app.NotificationCompatBase$UnreadConversation$Factory {
+  public abstract android.support.v4.app.NotificationCompatBase$UnreadConversation build(java.lang.String[], android.support.v4.app.RemoteInputCompatBase$RemoteInput, android.app.PendingIntent, android.app.PendingIntent, java.lang.String[], long);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatIceCreamSandwich$Builder support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatIceCreamSandwich$Builder
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatIceCreamSandwich$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatIceCreamSandwich$Builder	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v4.app.NotificationCompatIceCreamSandwich$Builder implements android.support.v4.app.NotificationBuilderWithBuilderAccessor {
+  public android.support.v4.app.NotificationCompatIceCreamSandwich$Builder(android.content.Context, android.app.Notification, java.lang.CharSequence, java.lang.CharSequence, java.lang.CharSequence, android.widget.RemoteViews, int, android.app.PendingIntent, android.app.PendingIntent, android.graphics.Bitmap, int, int, boolean);
+  public android.app.Notification$Builder getBuilder();
+  public android.app.Notification build();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatJellybean$Builder support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatJellybean$Builder
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatJellybean$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatJellybean$Builder	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.app.NotificationCompatJellybean$Builder implements android.support.v4.app.NotificationBuilderWithBuilderAccessor,android.support.v4.app.NotificationBuilderWithActions {
+  public android.support.v4.app.NotificationCompatJellybean$Builder(android.content.Context, android.app.Notification, java.lang.CharSequence, java.lang.CharSequence, java.lang.CharSequence, android.widget.RemoteViews, int, android.app.PendingIntent, android.app.PendingIntent, android.graphics.Bitmap, int, int, boolean, boolean, int, java.lang.CharSequence, boolean, android.os.Bundle, java.lang.String, boolean, java.lang.String, android.widget.RemoteViews, android.widget.RemoteViews);
+  public void addAction(android.support.v4.app.NotificationCompatBase$Action);
+  public android.app.Notification$Builder getBuilder();
+  public android.app.Notification build();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatKitKat$Builder support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatKitKat$Builder
--- support-compat-25.0.1_58db3096/android.support.v4.app.NotificationCompatKitKat$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.NotificationCompatKitKat$Builder	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.app.NotificationCompatKitKat$Builder implements android.support.v4.app.NotificationBuilderWithBuilderAccessor,android.support.v4.app.NotificationBuilderWithActions {
+  public android.support.v4.app.NotificationCompatKitKat$Builder(android.content.Context, android.app.Notification, java.lang.CharSequence, java.lang.CharSequence, java.lang.CharSequence, android.widget.RemoteViews, int, android.app.PendingIntent, android.app.PendingIntent, android.graphics.Bitmap, int, int, boolean, boolean, boolean, int, java.lang.CharSequence, boolean, java.util.ArrayList<java.lang.String>, android.os.Bundle, java.lang.String, boolean, java.lang.String, android.widget.RemoteViews, android.widget.RemoteViews);
+  public void addAction(android.support.v4.app.NotificationCompatBase$Action);
+  public android.app.Notification$Builder getBuilder();
+  public android.app.Notification build();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.RemoteInputCompatBase$RemoteInput support-compat-25.1.0_eca2886e/android.support.v4.app.RemoteInputCompatBase$RemoteInput
--- support-compat-25.0.1_58db3096/android.support.v4.app.RemoteInputCompatBase$RemoteInput	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.RemoteInputCompatBase$RemoteInput	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public abstract class android.support.v4.app.RemoteInputCompatBase$RemoteInput {
+  public android.support.v4.app.RemoteInputCompatBase$RemoteInput();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.RemoteInputCompatBase$RemoteInput$Factory support-compat-25.1.0_eca2886e/android.support.v4.app.RemoteInputCompatBase$RemoteInput$Factory
--- support-compat-25.0.1_58db3096/android.support.v4.app.RemoteInputCompatBase$RemoteInput$Factory	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.RemoteInputCompatBase$RemoteInput$Factory	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public interface android.support.v4.app.RemoteInputCompatBase$RemoteInput$Factory {
+  public abstract android.support.v4.app.RemoteInputCompatBase$RemoteInput build(java.lang.String, java.lang.CharSequence, java.lang.CharSequence[], boolean, android.os.Bundle);
+  public abstract android.support.v4.app.RemoteInputCompatBase$RemoteInput[] newArray(int);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.SupportActivity support-compat-25.1.0_eca2886e/android.support.v4.app.SupportActivity
--- support-compat-25.0.1_58db3096/android.support.v4.app.SupportActivity	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.SupportActivity	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v4.app.SupportActivity extends android.app.Activity {
+  public android.support.v4.app.SupportActivity();
+  public void putExtraData(android.support.v4.app.SupportActivity$ExtraData);
+  public <T extends android.support.v4.app.SupportActivity$ExtraData> T getExtraData(java.lang.Class<T>);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.app.SupportActivity$ExtraData support-compat-25.1.0_eca2886e/android.support.v4.app.SupportActivity$ExtraData
--- support-compat-25.0.1_58db3096/android.support.v4.app.SupportActivity$ExtraData	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.app.SupportActivity$ExtraData	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public class android.support.v4.app.SupportActivity$ExtraData {
+  public android.support.v4.app.SupportActivity$ExtraData();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.graphics.drawable.DrawableWrapper support-compat-25.1.0_eca2886e/android.support.v4.graphics.drawable.DrawableWrapper
--- support-compat-25.0.1_58db3096/android.support.v4.graphics.drawable.DrawableWrapper	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.graphics.drawable.DrawableWrapper	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public interface android.support.v4.graphics.drawable.DrawableWrapper {
+  public abstract android.graphics.drawable.Drawable getWrappedDrawable();
+  public abstract void setWrappedDrawable(android.graphics.drawable.Drawable);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.graphics.drawable.DrawableWrapperGingerbread$DrawableWrapperState support-compat-25.1.0_eca2886e/android.support.v4.graphics.drawable.DrawableWrapperGingerbread$DrawableWrapperState
--- support-compat-25.0.1_58db3096/android.support.v4.graphics.drawable.DrawableWrapperGingerbread$DrawableWrapperState	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.graphics.drawable.DrawableWrapperGingerbread$DrawableWrapperState	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,5 @@
+public abstract class android.support.v4.graphics.drawable.DrawableWrapperGingerbread$DrawableWrapperState extends android.graphics.drawable.Drawable$ConstantState {
+  public android.graphics.drawable.Drawable newDrawable();
+  public abstract android.graphics.drawable.Drawable newDrawable(android.content.res.Resources);
+  public int getChangingConfigurations();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.graphics.drawable.TintAwareDrawable support-compat-25.1.0_eca2886e/android.support.v4.graphics.drawable.TintAwareDrawable
--- support-compat-25.0.1_58db3096/android.support.v4.graphics.drawable.TintAwareDrawable	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.graphics.drawable.TintAwareDrawable	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,5 @@
+public interface android.support.v4.graphics.drawable.TintAwareDrawable {
+  public abstract void setTint(int);
+  public abstract void setTintList(android.content.res.ColorStateList);
+  public abstract void setTintMode(android.graphics.PorterDuff$Mode);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23 support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23
--- support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public final class android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23 {
+  public android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23();
+  public static boolean hasEnrolledFingerprints(android.content.Context);
+  public static boolean isHardwareDetected(android.content.Context);
+  public static void authenticate(android.content.Context, android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject, int, java.lang.Object, android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationCallback, android.os.Handler);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationCallback support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationCallback
--- support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationCallback	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationCallback	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,7 @@
+public abstract class android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationCallback {
+  public android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationCallback();
+  public void onAuthenticationError(int, java.lang.CharSequence);
+  public void onAuthenticationHelp(int, java.lang.CharSequence);
+  public void onAuthenticationSucceeded(android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationResultInternal);
+  public void onAuthenticationFailed();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationResultInternal support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationResultInternal
--- support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationResultInternal	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationResultInternal	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public final class android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationResultInternal {
+  public android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$AuthenticationResultInternal(android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject);
+  public android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject getCryptoObject();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject
--- support-compat-25.0.1_58db3096/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,8 @@
+public class android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject {
+  public android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject(java.security.Signature);
+  public android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject(javax.crypto.Cipher);
+  public android.support.v4.hardware.fingerprint.FingerprintManagerCompatApi23$CryptoObject(javax.crypto.Mac);
+  public java.security.Signature getSignature();
+  public javax.crypto.Cipher getCipher();
+  public javax.crypto.Mac getMac();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.net.TrafficStatsCompatApi24 support-compat-25.1.0_eca2886e/android.support.v4.net.TrafficStatsCompatApi24
--- support-compat-25.0.1_58db3096/android.support.v4.net.TrafficStatsCompatApi24	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.net.TrafficStatsCompatApi24	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v4.net.TrafficStatsCompatApi24 {
+  public android.support.v4.net.TrafficStatsCompatApi24();
+  public static void tagDatagramSocket(java.net.DatagramSocket) throws java.net.SocketException;
+  public static void untagDatagramSocket(java.net.DatagramSocket) throws java.net.SocketException;
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.os.BuildCompat support-compat-25.1.0_eca2886e/android.support.v4.os.BuildCompat
--- support-compat-25.0.1_58db3096/android.support.v4.os.BuildCompat	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.os.BuildCompat	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v4.os.BuildCompat {
+  public static boolean isAtLeastN();
+  public static boolean isAtLeastNMR1();
+  public static boolean isAtLeastO();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.os.ParcelableCompatCreatorCallbacks<T> support-compat-25.1.0_eca2886e/android.support.v4.os.ParcelableCompatCreatorCallbacks<T>
--- support-compat-25.0.1_58db3096/android.support.v4.os.ParcelableCompatCreatorCallbacks<T>	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.os.ParcelableCompatCreatorCallbacks<T>	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public interface android.support.v4.os.ParcelableCompatCreatorCallbacks<T> {
+  public abstract T createFromParcel(android.os.Parcel, java.lang.ClassLoader);
+  public abstract T[] newArray(int);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.os.UserManagerCompat support-compat-25.1.0_eca2886e/android.support.v4.os.UserManagerCompat
--- support-compat-25.0.1_58db3096/android.support.v4.os.UserManagerCompat	2016-12-15 02:15:53.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.os.UserManagerCompat	2016-12-15 02:15:53.000000000 +0900
@@ -2,2 +1,0 @@
-  public static boolean isUserRunningAndLocked(android.content.Context);
-  public static boolean isUserRunningAndUnlocked(android.content.Context);
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.os.UserManagerCompatApi24 support-compat-25.1.0_eca2886e/android.support.v4.os.UserManagerCompatApi24
--- support-compat-25.0.1_58db3096/android.support.v4.os.UserManagerCompatApi24	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.os.UserManagerCompatApi24	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v4.os.UserManagerCompatApi24 {
+  public android.support.v4.os.UserManagerCompatApi24();
+  public static boolean isUserUnlocked(android.content.Context);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.util.ArraySet<E> support-compat-25.1.0_eca2886e/android.support.v4.util.ArraySet<E>
--- support-compat-25.0.1_58db3096/android.support.v4.util.ArraySet<E>	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.util.ArraySet<E>	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,30 @@
+public final class android.support.v4.util.ArraySet<E> implements java.util.Collection<E>, java.util.Set<E> {
+  public android.support.v4.util.ArraySet();
+  public android.support.v4.util.ArraySet(int);
+  public android.support.v4.util.ArraySet(int, boolean);
+  public android.support.v4.util.ArraySet(android.support.v4.util.ArraySet<E>);
+  public android.support.v4.util.ArraySet(java.util.Collection<E>);
+  public void clear();
+  public void ensureCapacity(int);
+  public boolean contains(java.lang.Object);
+  public int indexOf(java.lang.Object);
+  public E valueAt(int);
+  public boolean isEmpty();
+  public boolean add(E);
+  public void append(E);
+  public void addAll(android.support.v4.util.ArraySet<? extends E>);
+  public boolean remove(java.lang.Object);
+  public E removeAt(int);
+  public boolean removeAll(android.support.v4.util.ArraySet<? extends E>);
+  public int size();
+  public java.lang.Object[] toArray();
+  public <T> T[] toArray(T[]);
+  public boolean equals(java.lang.Object);
+  public int hashCode();
+  public java.lang.String toString();
+  public java.util.Iterator<E> iterator();
+  public boolean containsAll(java.util.Collection<?>);
+  public boolean addAll(java.util.Collection<? extends E>);
+  public boolean removeAll(java.util.Collection<?>);
+  public boolean retainAll(java.util.Collection<?>);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.AccessibilityDelegateCompatIcs$AccessibilityDelegateBridge support-compat-25.1.0_eca2886e/android.support.v4.view.AccessibilityDelegateCompatIcs$AccessibilityDelegateBridge
--- support-compat-25.0.1_58db3096/android.support.v4.view.AccessibilityDelegateCompatIcs$AccessibilityDelegateBridge	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.AccessibilityDelegateCompatIcs$AccessibilityDelegateBridge	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,9 @@
+public interface android.support.v4.view.AccessibilityDelegateCompatIcs$AccessibilityDelegateBridge {
+  public abstract boolean dispatchPopulateAccessibilityEvent(android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract void onInitializeAccessibilityEvent(android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract void onInitializeAccessibilityNodeInfo(android.view.View, java.lang.Object);
+  public abstract void onPopulateAccessibilityEvent(android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract boolean onRequestSendAccessibilityEvent(android.view.ViewGroup, android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract void sendAccessibilityEvent(android.view.View, int);
+  public abstract void sendAccessibilityEventUnchecked(android.view.View, android.view.accessibility.AccessibilityEvent);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.AccessibilityDelegateCompatJellyBean$AccessibilityDelegateBridgeJellyBean support-compat-25.1.0_eca2886e/android.support.v4.view.AccessibilityDelegateCompatJellyBean$AccessibilityDelegateBridgeJellyBean
--- support-compat-25.0.1_58db3096/android.support.v4.view.AccessibilityDelegateCompatJellyBean$AccessibilityDelegateBridgeJellyBean	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.AccessibilityDelegateCompatJellyBean$AccessibilityDelegateBridgeJellyBean	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,11 @@
+public interface android.support.v4.view.AccessibilityDelegateCompatJellyBean$AccessibilityDelegateBridgeJellyBean {
+  public abstract boolean dispatchPopulateAccessibilityEvent(android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract void onInitializeAccessibilityEvent(android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract void onInitializeAccessibilityNodeInfo(android.view.View, java.lang.Object);
+  public abstract void onPopulateAccessibilityEvent(android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract boolean onRequestSendAccessibilityEvent(android.view.ViewGroup, android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract void sendAccessibilityEvent(android.view.View, int);
+  public abstract void sendAccessibilityEventUnchecked(android.view.View, android.view.accessibility.AccessibilityEvent);
+  public abstract java.lang.Object getAccessibilityNodeProvider(android.view.View);
+  public abstract boolean performAccessibilityAction(android.view.View, int, android.os.Bundle);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.LayoutInflaterFactory support-compat-25.1.0_eca2886e/android.support.v4.view.LayoutInflaterFactory
--- support-compat-25.0.1_58db3096/android.support.v4.view.LayoutInflaterFactory	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.LayoutInflaterFactory	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.view.LayoutInflaterFactory {
+  public abstract android.view.View onCreateView(android.view.View, java.lang.String, android.content.Context, android.util.AttributeSet);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.TintableBackgroundView support-compat-25.1.0_eca2886e/android.support.v4.view.TintableBackgroundView
--- support-compat-25.0.1_58db3096/android.support.v4.view.TintableBackgroundView	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.TintableBackgroundView	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public interface android.support.v4.view.TintableBackgroundView {
+  public abstract void setSupportBackgroundTintList(android.content.res.ColorStateList);
+  public abstract android.content.res.ColorStateList getSupportBackgroundTintList();
+  public abstract void setSupportBackgroundTintMode(android.graphics.PorterDuff$Mode);
+  public abstract android.graphics.PorterDuff$Mode getSupportBackgroundTintMode();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.ViewCompatLollipop$OnApplyWindowInsetsListenerBridge support-compat-25.1.0_eca2886e/android.support.v4.view.ViewCompatLollipop$OnApplyWindowInsetsListenerBridge
--- support-compat-25.0.1_58db3096/android.support.v4.view.ViewCompatLollipop$OnApplyWindowInsetsListenerBridge	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.ViewCompatLollipop$OnApplyWindowInsetsListenerBridge	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.view.ViewCompatLollipop$OnApplyWindowInsetsListenerBridge {
+  public abstract java.lang.Object onApplyWindowInsets(android.view.View, java.lang.Object);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.ViewPropertyAnimatorListener support-compat-25.1.0_eca2886e/android.support.v4.view.ViewPropertyAnimatorListener
--- support-compat-25.0.1_58db3096/android.support.v4.view.ViewPropertyAnimatorListener	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.ViewPropertyAnimatorListener	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,5 @@
+public interface android.support.v4.view.ViewPropertyAnimatorListener {
+  public abstract void onAnimationStart(android.view.View);
+  public abstract void onAnimationEnd(android.view.View);
+  public abstract void onAnimationCancel(android.view.View);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.ViewPropertyAnimatorListenerAdapter support-compat-25.1.0_eca2886e/android.support.v4.view.ViewPropertyAnimatorListenerAdapter
--- support-compat-25.0.1_58db3096/android.support.v4.view.ViewPropertyAnimatorListenerAdapter	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.ViewPropertyAnimatorListenerAdapter	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.view.ViewPropertyAnimatorListenerAdapter implements android.support.v4.view.ViewPropertyAnimatorListener {
+  public android.support.v4.view.ViewPropertyAnimatorListenerAdapter();
+  public void onAnimationStart(android.view.View);
+  public void onAnimationEnd(android.view.View);
+  public void onAnimationCancel(android.view.View);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.ViewPropertyAnimatorUpdateListener support-compat-25.1.0_eca2886e/android.support.v4.view.ViewPropertyAnimatorUpdateListener
--- support-compat-25.0.1_58db3096/android.support.v4.view.ViewPropertyAnimatorUpdateListener	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.ViewPropertyAnimatorUpdateListener	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.view.ViewPropertyAnimatorUpdateListener {
+  public abstract void onAnimationUpdate(android.view.View);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.accessibility.AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerWrapper support-compat-25.1.0_eca2886e/android.support.v4.view.accessibility.AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerWrapper
--- support-compat-25.0.1_58db3096/android.support.v4.view.accessibility.AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerWrapper	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.accessibility.AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerWrapper	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.view.accessibility.AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerWrapper implements android.view.accessibility.AccessibilityManager$AccessibilityStateChangeListener {
+  public android.support.v4.view.accessibility.AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerWrapper(java.lang.Object, android.support.v4.view.accessibility.AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerBridge);
+  public int hashCode();
+  public boolean equals(java.lang.Object);
+  public void onAccessibilityStateChanged(boolean);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.accessibility.AccessibilityManagerCompatKitKat$TouchExplorationStateChangeListenerWrapper support-compat-25.1.0_eca2886e/android.support.v4.view.accessibility.AccessibilityManagerCompatKitKat$TouchExplorationStateChangeListenerWrapper
--- support-compat-25.0.1_58db3096/android.support.v4.view.accessibility.AccessibilityManagerCompatKitKat$TouchExplorationStateChangeListenerWrapper	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.accessibility.AccessibilityManagerCompatKitKat$TouchExplorationStateChangeListenerWrapper	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v4.view.accessibility.AccessibilityManagerCompatKitKat$TouchExplorationStateChangeListenerWrapper implements android.view.accessibility.AccessibilityManager$TouchExplorationStateChangeListener {
+  public android.support.v4.view.accessibility.AccessibilityManagerCompatKitKat$TouchExplorationStateChangeListenerWrapper(java.lang.Object, android.support.v4.view.accessibility.AccessibilityManagerCompatKitKat$TouchExplorationStateChangeListenerBridge);
+  public int hashCode();
+  public boolean equals(java.lang.Object);
+  public void onTouchExplorationStateChanged(boolean);
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat support-compat-25.1.0_eca2886e/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat
--- support-compat-25.0.1_58db3096/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat	2016-12-15 02:15:53.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat	2016-12-15 02:15:53.000000000 +0900
@@ -1,0 +2 @@
+  public int mParentVirtualDescendantId;
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.widget.SearchViewCompatIcs$MySearchView support-compat-25.1.0_eca2886e/android.support.v4.widget.SearchViewCompatIcs$MySearchView
--- support-compat-25.0.1_58db3096/android.support.v4.widget.SearchViewCompatIcs$MySearchView	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.widget.SearchViewCompatIcs$MySearchView	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v4.widget.SearchViewCompatIcs$MySearchView extends android.widget.SearchView {
+  public android.support.v4.widget.SearchViewCompatIcs$MySearchView(android.content.Context);
+  public void onActionViewCollapsed();
+}
diff -U 0 -N support-compat-25.0.1_58db3096/android.support.v4.widget.TintableCompoundButton support-compat-25.1.0_eca2886e/android.support.v4.widget.TintableCompoundButton
--- support-compat-25.0.1_58db3096/android.support.v4.widget.TintableCompoundButton	1970-01-01 09:00:00.000000000 +0900
+++ support-compat-25.1.0_eca2886e/android.support.v4.widget.TintableCompoundButton	2016-12-15 02:15:53.000000000 +0900
@@ -0,0 +1,6 @@
+public interface android.support.v4.widget.TintableCompoundButton {
+  public abstract void setSupportButtonTintList(android.content.res.ColorStateList);
+  public abstract android.content.res.ColorStateList getSupportButtonTintList();
+  public abstract void setSupportButtonTintMode(android.graphics.PorterDuff$Mode);
+  public abstract android.graphics.PorterDuff$Mode getSupportButtonTintMode();
+}
