diff -U 0 -N preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.Preference preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.Preference
--- preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.Preference	2016-05-16 18:10:00.000000000 +0900
+++ preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.Preference	2016-05-16 18:11:04.000000000 +0900
@@ -19,0 +20 @@
+  public void setViewId(int);
@@ -63,0 +65 @@
+  public void onInitializeAccessibilityNodeInfo(android.support.v4.view.accessibility.AccessibilityNodeInfoCompat);
diff -U 0 -N preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceCategory preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceCategory
--- preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceCategory	2016-05-16 18:10:00.000000000 +0900
+++ preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceCategory	2016-05-16 18:11:04.000000000 +0900
@@ -7,0 +8 @@
+  public void onInitializeAccessibilityNodeInfo(android.support.v4.view.accessibility.AccessibilityNodeInfoCompat);
diff -U 0 -N preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceFragmentCompat preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceFragmentCompat
--- preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceFragmentCompat	2016-05-16 18:10:00.000000000 +0900
+++ preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceFragmentCompat	2016-05-16 18:11:04.000000000 +0900
@@ -27,0 +28,2 @@
+  public void scrollToPreference(java.lang.String);
+  public void scrollToPreference(android.support.v7.preference.Preference);
diff -U 0 -N preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceGroup$PreferencePositionCallback preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceGroup$PreferencePositionCallback
--- preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceGroup$PreferencePositionCallback	1970-01-01 09:00:00.000000000 +0900
+++ preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceGroup$PreferencePositionCallback	2016-05-16 18:11:04.000000000 +0900
@@ -0,0 +1,4 @@
+public interface android.support.v7.preference.PreferenceGroup$PreferencePositionCallback {
+  public abstract int getPreferenceAdapterPosition(java.lang.String);
+  public abstract int getPreferenceAdapterPosition(android.support.v7.preference.Preference);
+}
diff -U 0 -N preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceGroupAdapter preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceGroupAdapter
--- preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceGroupAdapter	2016-05-16 18:10:00.000000000 +0900
+++ preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceGroupAdapter	2016-05-16 18:11:04.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.preference.PreferenceGroupAdapter extends android.support.v7.widget.RecyclerView$Adapter<android.support.v7.preference.PreferenceViewHolder> implements android.support.v7.preference.Preference$OnPreferenceChangeInternalListener {
+public class android.support.v7.preference.PreferenceGroupAdapter extends android.support.v7.widget.RecyclerView$Adapter<android.support.v7.preference.PreferenceViewHolder> implements android.support.v7.preference.Preference$OnPreferenceChangeInternalListener, android.support.v7.preference.PreferenceGroup$PreferencePositionCallback {
@@ -11,0 +12,2 @@
+  public int getPreferenceAdapterPosition(java.lang.String);
+  public int getPreferenceAdapterPosition(android.support.v7.preference.Preference);
diff -U 0 -N preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceRecyclerViewAccessibilityDelegate preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceRecyclerViewAccessibilityDelegate
--- preference-v7-24.0.0-alpha2_203e405e/android.support.v7.preference.PreferenceRecyclerViewAccessibilityDelegate	1970-01-01 09:00:00.000000000 +0900
+++ preference-v7-24.0.0-beta1_b50e144e/android.support.v7.preference.PreferenceRecyclerViewAccessibilityDelegate	2016-05-16 18:11:04.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v7.preference.PreferenceRecyclerViewAccessibilityDelegate extends android.support.v7.widget.RecyclerViewAccessibilityDelegate {
+  public android.support.v7.preference.PreferenceRecyclerViewAccessibilityDelegate(android.support.v7.widget.RecyclerView);
+  public android.support.v4.view.AccessibilityDelegateCompat getItemDelegate();
+}
