diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.app.ActionBar appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBar
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.app.ActionBar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBar	2015-08-20 23:41:44.000000000 +0900
@@ -80,0 +81 @@
+  public boolean onKeyShortcut(int, android.view.KeyEvent);
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.app.ActionBarActivity appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBarActivity
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.app.ActionBarActivity	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBarActivity	2015-08-20 23:41:44.000000000 +0900
@@ -11 +10,0 @@
-  public android.view.View onCreatePanelView(int);
@@ -37 +36 @@
-  public boolean onKeyDown(int, android.view.KeyEvent);
+  public boolean dispatchKeyEvent(android.view.KeyEvent);
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.app.ActionBarDrawerToggle$Delegate appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBarDrawerToggle$Delegate
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.app.ActionBarDrawerToggle$Delegate	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBarDrawerToggle$Delegate	2015-08-20 23:41:44.000000000 +0900
@@ -5,0 +6 @@
+  public abstract boolean isNavigationVisible();
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.appcompat.BuildConfig appcompat-v7-22.0.0_6bfba991/android.support.v7.appcompat.BuildConfig
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.appcompat.BuildConfig	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.appcompat.BuildConfig	2015-08-20 23:41:44.000000000 +0900
@@ -3 +3 @@
-  public static final java.lang.String PACKAGE_NAME;
+  public static final java.lang.String APPLICATION_ID;
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.app.ToolbarActionBar appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.ToolbarActionBar
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.app.ToolbarActionBar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.ToolbarActionBar	2015-08-20 23:41:44.000000000 +0900
@@ -68,0 +69 @@
+  public boolean onKeyShortcut(int, android.view.KeyEvent);
@@ -72 +72,0 @@
-  public void setListMenuPresenter(android.support.v7.internal.view.menu.ListMenuPresenter);
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.app.WindowDecorActionBar$ActionModeImpl appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.WindowDecorActionBar$ActionModeImpl
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.app.WindowDecorActionBar$ActionModeImpl	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.WindowDecorActionBar$ActionModeImpl	2015-08-20 23:41:44.000000000 +0900
@@ -2 +2 @@
-  public android.support.v7.internal.app.WindowDecorActionBar$ActionModeImpl(android.support.v7.internal.app.WindowDecorActionBar, android.support.v7.view.ActionMode$Callback);
+  public android.support.v7.internal.app.WindowDecorActionBar$ActionModeImpl(android.support.v7.internal.app.WindowDecorActionBar, android.content.Context, android.support.v7.view.ActionMode$Callback);
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.ActivityChooserView$InnerLayout appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ActivityChooserView$InnerLayout
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.ActivityChooserView$InnerLayout	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ActivityChooserView$InnerLayout	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,3 @@
+public class android.support.v7.internal.widget.ActivityChooserView$InnerLayout extends android.support.v7.widget.LinearLayoutCompat {
+  public android.support.v7.internal.widget.ActivityChooserView$InnerLayout(android.content.Context, android.util.AttributeSet);
+}
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.DecorToolbar appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DecorToolbar
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.DecorToolbar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DecorToolbar	2015-08-20 23:41:44.000000000 +0900
@@ -56,0 +57,7 @@
+  public abstract void setBackgroundDrawable(android.graphics.drawable.Drawable);
+  public abstract int getHeight();
+  public abstract void setVisibility(int);
+  public abstract int getVisibility();
+  public abstract void setMenuCallbacks(android.support.v7.internal.view.menu.MenuPresenter$Callback, android.support.v7.internal.view.menu.MenuBuilder$Callback);
+  public abstract android.view.Menu getMenu();
+  public abstract int getPopupTheme();
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintAutoCompleteTextView appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintAutoCompleteTextView
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintAutoCompleteTextView	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintAutoCompleteTextView	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v7.internal.widget.TintAutoCompleteTextView extends android.widget.AutoCompleteTextView {
+  public android.support.v7.internal.widget.TintAutoCompleteTextView(android.content.Context);
+  public android.support.v7.internal.widget.TintAutoCompleteTextView(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.internal.widget.TintAutoCompleteTextView(android.content.Context, android.util.AttributeSet, int);
+  public void setDropDownBackgroundResource(int);
+}
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintButton appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintButton
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintButton	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintButton	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v7.internal.widget.TintButton extends android.widget.Button {
+  public android.support.v7.internal.widget.TintButton(android.content.Context);
+  public android.support.v7.internal.widget.TintButton(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.internal.widget.TintButton(android.content.Context, android.util.AttributeSet, int);
+  public void setBackgroundResource(int);
+}
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintMultiAutoCompleteTextView appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintMultiAutoCompleteTextView
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintMultiAutoCompleteTextView	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintMultiAutoCompleteTextView	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v7.internal.widget.TintMultiAutoCompleteTextView extends android.widget.MultiAutoCompleteTextView {
+  public android.support.v7.internal.widget.TintMultiAutoCompleteTextView(android.content.Context);
+  public android.support.v7.internal.widget.TintMultiAutoCompleteTextView(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.internal.widget.TintMultiAutoCompleteTextView(android.content.Context, android.util.AttributeSet, int);
+  public void setDropDownBackgroundResource(int);
+}
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintRatingBar appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintRatingBar
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.TintRatingBar	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintRatingBar	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v7.internal.widget.TintRatingBar extends android.widget.RatingBar {
+  public android.support.v7.internal.widget.TintRatingBar(android.content.Context);
+  public android.support.v7.internal.widget.TintRatingBar(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.internal.widget.TintRatingBar(android.content.Context, android.util.AttributeSet, int);
+}
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.ToolbarWidgetWrapper appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ToolbarWidgetWrapper
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.internal.widget.ToolbarWidgetWrapper	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ToolbarWidgetWrapper	2015-08-20 23:41:44.000000000 +0900
@@ -58,0 +59,7 @@
+  public void setBackgroundDrawable(android.graphics.drawable.Drawable);
+  public int getHeight();
+  public void setVisibility(int);
+  public int getVisibility();
+  public void setMenuCallbacks(android.support.v7.internal.view.menu.MenuPresenter$Callback, android.support.v7.internal.view.menu.MenuBuilder$Callback);
+  public android.view.Menu getMenu();
+  public int getPopupTheme();
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.widget.SearchView$SearchAutoComplete appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.SearchView$SearchAutoComplete
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.widget.SearchView$SearchAutoComplete	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.SearchView$SearchAutoComplete	2015-08-20 23:41:44.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.widget.SearchView$SearchAutoComplete extends android.widget.AutoCompleteTextView {
+public class android.support.v7.widget.SearchView$SearchAutoComplete extends android.support.v7.internal.widget.TintAutoCompleteTextView {
@@ -6 +5,0 @@
-  public void setDropDownBackgroundResource(int);
diff -U 0 -N appcompat-v7-21.0.3_4f967e6f/android.support.v7.widget.Toolbar appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.Toolbar
--- appcompat-v7-21.0.3_4f967e6f/android.support.v7.widget.Toolbar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.Toolbar	2015-08-20 23:41:44.000000000 +0900
@@ -50,0 +51 @@
+  public boolean onHoverEvent(android.view.MotionEvent);
