diff -U 0 -N appcompat-v7-21.0.0_8db30102/android.support.v7.internal.view.menu.MenuItemWrapperICS appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.view.menu.MenuItemWrapperICS
--- appcompat-v7-21.0.0_8db30102/android.support.v7.internal.view.menu.MenuItemWrapperICS	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.view.menu.MenuItemWrapperICS	2015-08-20 23:41:44.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.internal.view.menu.MenuItemWrapperICS extends android.support.v7.internal.view.menu.BaseMenuWrapper<android.view.MenuItem> implements android.support.v4.internal.view.SupportMenuItem {
+public class android.support.v7.internal.view.menu.MenuItemWrapperICS extends android.support.v7.internal.view.menu.BaseMenuWrapper<android.support.v4.internal.view.SupportMenuItem> implements android.view.MenuItem {
@@ -43,3 +42,0 @@
-  public android.support.v4.internal.view.SupportMenuItem setSupportOnActionExpandListener(android.support.v4.view.MenuItemCompat$OnActionExpandListener);
-  public android.support.v4.internal.view.SupportMenuItem setSupportActionProvider(android.support.v4.view.ActionProvider);
-  public android.support.v4.view.ActionProvider getSupportActionProvider();
diff -U 0 -N appcompat-v7-21.0.0_8db30102/android.support.v7.internal.view.menu.MenuPopupHelper appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.view.menu.MenuPopupHelper
--- appcompat-v7-21.0.0_8db30102/android.support.v7.internal.view.menu.MenuPopupHelper	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.view.menu.MenuPopupHelper	2015-08-20 23:41:44.000000000 +0900
@@ -4,0 +5 @@
+  public android.support.v7.internal.view.menu.MenuPopupHelper(android.content.Context, android.support.v7.internal.view.menu.MenuBuilder, android.view.View, boolean, int, int);
diff -U 0 -N appcompat-v7-21.0.0_8db30102/android.support.v7.internal.view.menu.MenuWrapperFactory appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.view.menu.MenuWrapperFactory
--- appcompat-v7-21.0.0_8db30102/android.support.v7.internal.view.menu.MenuWrapperFactory	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.view.menu.MenuWrapperFactory	2015-08-20 23:41:44.000000000 +0900
@@ -2,5 +2,3 @@
-  public static android.view.Menu createMenuWrapper(android.view.Menu);
-  public static android.view.MenuItem createMenuItemWrapper(android.view.MenuItem);
-  public static android.support.v4.internal.view.SupportMenu createSupportMenuWrapper(android.view.Menu);
-  public static android.support.v4.internal.view.SupportSubMenu createSupportSubMenuWrapper(android.view.SubMenu);
-  public static android.support.v4.internal.view.SupportMenuItem createSupportMenuItemWrapper(android.view.MenuItem);
+  public static android.view.Menu wrapSupportMenu(android.content.Context, android.support.v4.internal.view.SupportMenu);
+  public static android.view.MenuItem wrapSupportMenuItem(android.content.Context, android.support.v4.internal.view.SupportMenuItem);
+  public static android.view.SubMenu wrapSupportSubMenu(android.content.Context, android.support.v4.internal.view.SupportSubMenu);
diff -U 0 -N appcompat-v7-21.0.0_8db30102/android.support.v7.internal.widget.ProgressBarCompat appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.widget.ProgressBarCompat
--- appcompat-v7-21.0.0_8db30102/android.support.v7.internal.widget.ProgressBarCompat	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.internal.widget.ProgressBarCompat	1970-01-01 09:00:00.000000000 +0900
@@ -1,25 +0,0 @@
-public class android.support.v7.internal.widget.ProgressBarCompat extends android.view.View {
-  public android.support.v7.internal.widget.ProgressBarCompat(android.content.Context, android.util.AttributeSet, int, int);
-  public synchronized boolean isIndeterminate();
-  public synchronized void setIndeterminate(boolean);
-  public android.graphics.drawable.Drawable getIndeterminateDrawable();
-  public void setIndeterminateDrawable(android.graphics.drawable.Drawable);
-  public android.graphics.drawable.Drawable getProgressDrawable();
-  public void setProgressDrawable(android.graphics.drawable.Drawable);
-  public void postInvalidate();
-  public synchronized void setProgress(int);
-  public synchronized void setSecondaryProgress(int);
-  public synchronized int getProgress();
-  public synchronized int getSecondaryProgress();
-  public synchronized int getMax();
-  public synchronized void setMax(int);
-  public final synchronized void incrementProgressBy(int);
-  public final synchronized void incrementSecondaryProgressBy(int);
-  public void setInterpolator(android.content.Context, int);
-  public void setInterpolator(android.view.animation.Interpolator);
-  public android.view.animation.Interpolator getInterpolator();
-  public void setVisibility(int);
-  public void invalidateDrawable(android.graphics.drawable.Drawable);
-  public android.os.Parcelable onSaveInstanceState();
-  public void onRestoreInstanceState(android.os.Parcelable);
-}
diff -U 0 -N appcompat-v7-21.0.0_8db30102/android.support.v7.widget.ListPopupWindow appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.widget.ListPopupWindow
--- appcompat-v7-21.0.0_8db30102/android.support.v7.widget.ListPopupWindow	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.widget.ListPopupWindow	2015-08-20 23:41:44.000000000 +0900
@@ -11,0 +12 @@
+  public android.support.v7.widget.ListPopupWindow(android.content.Context, android.util.AttributeSet, int, int);
diff -U 0 -N appcompat-v7-21.0.0_8db30102/android.support.v7.widget.PopupMenu appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.widget.PopupMenu
--- appcompat-v7-21.0.0_8db30102/android.support.v7.widget.PopupMenu	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-21.0.2_2cbdd7ce/android.support.v7.widget.PopupMenu	2015-08-20 23:41:44.000000000 +0900
@@ -3,0 +4 @@
+  public android.support.v7.widget.PopupMenu(android.content.Context, android.view.View, int, int, int);
