diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBarActivity appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.ActionBarActivity
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.ActionBarActivity	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.ActionBarActivity	2015-08-20 23:41:44.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.app.ActionBarActivity extends android.support.v4.app.FragmentActivity implements android.support.v7.app.ActionBar$Callback,android.support.v4.app.TaskStackBuilder$SupportParentable,android.support.v4.app.ActionBarDrawerToggle$DelegateProvider,android.support.v7.app.ActionBarDrawerToggle$TmpDelegateProvider {
+public class android.support.v7.app.ActionBarActivity extends android.support.v7.app.AppCompatActivity {
@@ -3,37 +2,0 @@
-  public android.support.v7.app.ActionBar getSupportActionBar();
-  public void setSupportActionBar(android.support.v7.widget.Toolbar);
-  public android.view.MenuInflater getMenuInflater();
-  public void setContentView(int);
-  public void setContentView(android.view.View);
-  public void setContentView(android.view.View, android.view.ViewGroup$LayoutParams);
-  public void addContentView(android.view.View, android.view.ViewGroup$LayoutParams);
-  public void onConfigurationChanged(android.content.res.Configuration);
-  public final boolean onMenuItemSelected(int, android.view.MenuItem);
-  public boolean supportRequestWindowFeature(int);
-  public void supportInvalidateOptionsMenu();
-  public void invalidateOptionsMenu();
-  public void onSupportActionModeStarted(android.support.v7.view.ActionMode);
-  public void onSupportActionModeFinished(android.support.v7.view.ActionMode);
-  public android.support.v7.view.ActionMode startSupportActionMode(android.support.v7.view.ActionMode$Callback);
-  public boolean onCreatePanelMenu(int, android.view.Menu);
-  public boolean onPreparePanel(int, android.view.View, android.view.Menu);
-  public void onPanelClosed(int, android.view.Menu);
-  public boolean onMenuOpened(int, android.view.Menu);
-  public void onBackPressed();
-  public void setSupportProgressBarVisibility(boolean);
-  public void setSupportProgressBarIndeterminateVisibility(boolean);
-  public void setSupportProgressBarIndeterminate(boolean);
-  public void setSupportProgress(int);
-  public void onCreateSupportNavigateUpTaskStack(android.support.v4.app.TaskStackBuilder);
-  public void onPrepareSupportNavigateUpTaskStack(android.support.v4.app.TaskStackBuilder);
-  public boolean onSupportNavigateUp();
-  public android.content.Intent getSupportParentActivityIntent();
-  public boolean supportShouldUpRecreateTask(android.content.Intent);
-  public void supportNavigateUpTo(android.content.Intent);
-  public final android.support.v4.app.ActionBarDrawerToggle$Delegate getDrawerToggleDelegate();
-  public android.support.v7.app.ActionBarDrawerToggle$Delegate getV7DrawerToggleDelegate();
-  public boolean onKeyShortcut(int, android.view.KeyEvent);
-  public boolean dispatchKeyEvent(android.view.KeyEvent);
-  public final void onContentChanged();
-  public void onSupportContentChanged();
-  public android.view.View onCreateView(java.lang.String, android.content.Context, android.util.AttributeSet);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertController$AlertParams appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertController$AlertParams
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertController$AlertParams	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertController$AlertParams	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,44 @@
+public class android.support.v7.app.AlertController$AlertParams {
+  public final android.content.Context mContext;
+  public final android.view.LayoutInflater mInflater;
+  public int mIconId;
+  public android.graphics.drawable.Drawable mIcon;
+  public int mIconAttrId;
+  public java.lang.CharSequence mTitle;
+  public android.view.View mCustomTitleView;
+  public java.lang.CharSequence mMessage;
+  public java.lang.CharSequence mPositiveButtonText;
+  public android.content.DialogInterface$OnClickListener mPositiveButtonListener;
+  public java.lang.CharSequence mNegativeButtonText;
+  public android.content.DialogInterface$OnClickListener mNegativeButtonListener;
+  public java.lang.CharSequence mNeutralButtonText;
+  public android.content.DialogInterface$OnClickListener mNeutralButtonListener;
+  public boolean mCancelable;
+  public android.content.DialogInterface$OnCancelListener mOnCancelListener;
+  public android.content.DialogInterface$OnDismissListener mOnDismissListener;
+  public android.content.DialogInterface$OnKeyListener mOnKeyListener;
+  public java.lang.CharSequence[] mItems;
+  public android.widget.ListAdapter mAdapter;
+  public android.content.DialogInterface$OnClickListener mOnClickListener;
+  public int mViewLayoutResId;
+  public android.view.View mView;
+  public int mViewSpacingLeft;
+  public int mViewSpacingTop;
+  public int mViewSpacingRight;
+  public int mViewSpacingBottom;
+  public boolean mViewSpacingSpecified;
+  public boolean[] mCheckedItems;
+  public boolean mIsMultiChoice;
+  public boolean mIsSingleChoice;
+  public int mCheckedItem;
+  public android.content.DialogInterface$OnMultiChoiceClickListener mOnCheckboxClickListener;
+  public android.database.Cursor mCursor;
+  public java.lang.String mLabelColumn;
+  public java.lang.String mIsCheckedColumn;
+  public boolean mForceInverseBackground;
+  public android.widget.AdapterView$OnItemSelectedListener mOnItemSelectedListener;
+  public android.support.v7.app.AlertController$AlertParams$OnPrepareListViewListener mOnPrepareListViewListener;
+  public boolean mRecycleOnMeasure;
+  public android.support.v7.app.AlertController$AlertParams(android.content.Context);
+  public void apply(android.support.v7.app.AlertController);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertController$AlertParams$OnPrepareListViewListener appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertController$AlertParams$OnPrepareListViewListener
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertController$AlertParams$OnPrepareListViewListener	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertController$AlertParams$OnPrepareListViewListener	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v7.app.AlertController$AlertParams$OnPrepareListViewListener {
+  public abstract void onPrepareListView(android.widget.ListView);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertDialog appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertDialog
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertDialog	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertDialog	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,16 @@
+public class android.support.v7.app.AlertDialog extends android.support.v7.app.AppCompatDialog implements android.content.DialogInterface {
+  public android.widget.Button getButton(int);
+  public android.widget.ListView getListView();
+  public void setTitle(java.lang.CharSequence);
+  public void setCustomTitle(android.view.View);
+  public void setMessage(java.lang.CharSequence);
+  public void setView(android.view.View);
+  public void setView(android.view.View, int, int, int, int);
+  public void setButton(int, java.lang.CharSequence, android.os.Message);
+  public void setButton(int, java.lang.CharSequence, android.content.DialogInterface$OnClickListener);
+  public void setIcon(int);
+  public void setIcon(android.graphics.drawable.Drawable);
+  public void setIconAttribute(int);
+  public boolean onKeyDown(int, android.view.KeyEvent);
+  public boolean onKeyUp(int, android.view.KeyEvent);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertDialog$Builder appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertDialog$Builder
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AlertDialog$Builder	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AlertDialog$Builder	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,42 @@
+public class android.support.v7.app.AlertDialog$Builder {
+  public android.support.v7.app.AlertDialog$Builder(android.content.Context);
+  public android.support.v7.app.AlertDialog$Builder(android.content.Context, int);
+  public android.content.Context getContext();
+  public android.support.v7.app.AlertDialog$Builder setTitle(int);
+  public android.support.v7.app.AlertDialog$Builder setTitle(java.lang.CharSequence);
+  public android.support.v7.app.AlertDialog$Builder setCustomTitle(android.view.View);
+  public android.support.v7.app.AlertDialog$Builder setMessage(int);
+  public android.support.v7.app.AlertDialog$Builder setMessage(java.lang.CharSequence);
+  public android.support.v7.app.AlertDialog$Builder setIcon(int);
+  public android.support.v7.app.AlertDialog$Builder setIcon(android.graphics.drawable.Drawable);
+  public android.support.v7.app.AlertDialog$Builder setIconAttribute(int);
+  public android.support.v7.app.AlertDialog$Builder setPositiveButton(int, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setPositiveButton(java.lang.CharSequence, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setNegativeButton(int, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setNegativeButton(java.lang.CharSequence, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setNeutralButton(int, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setNeutralButton(java.lang.CharSequence, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setCancelable(boolean);
+  public android.support.v7.app.AlertDialog$Builder setOnCancelListener(android.content.DialogInterface$OnCancelListener);
+  public android.support.v7.app.AlertDialog$Builder setOnDismissListener(android.content.DialogInterface$OnDismissListener);
+  public android.support.v7.app.AlertDialog$Builder setOnKeyListener(android.content.DialogInterface$OnKeyListener);
+  public android.support.v7.app.AlertDialog$Builder setItems(int, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setItems(java.lang.CharSequence[], android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setAdapter(android.widget.ListAdapter, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setCursor(android.database.Cursor, android.content.DialogInterface$OnClickListener, java.lang.String);
+  public android.support.v7.app.AlertDialog$Builder setMultiChoiceItems(int, boolean[], android.content.DialogInterface$OnMultiChoiceClickListener);
+  public android.support.v7.app.AlertDialog$Builder setMultiChoiceItems(java.lang.CharSequence[], boolean[], android.content.DialogInterface$OnMultiChoiceClickListener);
+  public android.support.v7.app.AlertDialog$Builder setMultiChoiceItems(android.database.Cursor, java.lang.String, java.lang.String, android.content.DialogInterface$OnMultiChoiceClickListener);
+  public android.support.v7.app.AlertDialog$Builder setSingleChoiceItems(int, int, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setSingleChoiceItems(android.database.Cursor, int, java.lang.String, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setSingleChoiceItems(java.lang.CharSequence[], int, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setSingleChoiceItems(android.widget.ListAdapter, int, android.content.DialogInterface$OnClickListener);
+  public android.support.v7.app.AlertDialog$Builder setOnItemSelectedListener(android.widget.AdapterView$OnItemSelectedListener);
+  public android.support.v7.app.AlertDialog$Builder setView(int);
+  public android.support.v7.app.AlertDialog$Builder setView(android.view.View);
+  public android.support.v7.app.AlertDialog$Builder setView(android.view.View, int, int, int, int);
+  public android.support.v7.app.AlertDialog$Builder setInverseBackgroundForced(boolean);
+  public android.support.v7.app.AlertDialog$Builder setRecycleOnMeasureEnabled(boolean);
+  public android.support.v7.app.AlertDialog create();
+  public android.support.v7.app.AlertDialog show();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatActivity appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatActivity
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatActivity	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatActivity	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,32 @@
+public class android.support.v7.app.AppCompatActivity extends android.support.v4.app.FragmentActivity implements android.support.v7.app.AppCompatCallback,android.support.v4.app.TaskStackBuilder$SupportParentable,android.support.v7.app.ActionBarDrawerToggle$DelegateProvider {
+  public android.support.v7.app.AppCompatActivity();
+  public android.support.v7.app.ActionBar getSupportActionBar();
+  public void setSupportActionBar(android.support.v7.widget.Toolbar);
+  public android.view.MenuInflater getMenuInflater();
+  public void setContentView(int);
+  public void setContentView(android.view.View);
+  public void setContentView(android.view.View, android.view.ViewGroup$LayoutParams);
+  public void addContentView(android.view.View, android.view.ViewGroup$LayoutParams);
+  public void onConfigurationChanged(android.content.res.Configuration);
+  public final boolean onMenuItemSelected(int, android.view.MenuItem);
+  public boolean supportRequestWindowFeature(int);
+  public void supportInvalidateOptionsMenu();
+  public void invalidateOptionsMenu();
+  public void onSupportActionModeStarted(android.support.v7.view.ActionMode);
+  public void onSupportActionModeFinished(android.support.v7.view.ActionMode);
+  public android.support.v7.view.ActionMode startSupportActionMode(android.support.v7.view.ActionMode$Callback);
+  public void setSupportProgressBarVisibility(boolean);
+  public void setSupportProgressBarIndeterminateVisibility(boolean);
+  public void setSupportProgressBarIndeterminate(boolean);
+  public void setSupportProgress(int);
+  public void onCreateSupportNavigateUpTaskStack(android.support.v4.app.TaskStackBuilder);
+  public void onPrepareSupportNavigateUpTaskStack(android.support.v4.app.TaskStackBuilder);
+  public boolean onSupportNavigateUp();
+  public android.content.Intent getSupportParentActivityIntent();
+  public boolean supportShouldUpRecreateTask(android.content.Intent);
+  public void supportNavigateUpTo(android.content.Intent);
+  public void onContentChanged();
+  public void onSupportContentChanged();
+  public android.support.v7.app.ActionBarDrawerToggle$Delegate getDrawerToggleDelegate();
+  public android.support.v7.app.AppCompatDelegate getDelegate();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatCallback appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatCallback
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatCallback	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatCallback	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,4 @@
+public interface android.support.v7.app.AppCompatCallback {
+  public abstract void onSupportActionModeStarted(android.support.v7.view.ActionMode);
+  public abstract void onSupportActionModeFinished(android.support.v7.view.ActionMode);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatDelegate appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatDelegate
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatDelegate	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatDelegate	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,24 @@
+public abstract class android.support.v7.app.AppCompatDelegate {
+  public static android.support.v7.app.AppCompatDelegate create(android.app.Activity, android.support.v7.app.AppCompatCallback);
+  public static android.support.v7.app.AppCompatDelegate create(android.app.Dialog, android.support.v7.app.AppCompatCallback);
+  public abstract android.support.v7.app.ActionBar getSupportActionBar();
+  public abstract void setSupportActionBar(android.support.v7.widget.Toolbar);
+  public abstract android.view.MenuInflater getMenuInflater();
+  public abstract void onCreate(android.os.Bundle);
+  public abstract void onPostCreate(android.os.Bundle);
+  public abstract void onConfigurationChanged(android.content.res.Configuration);
+  public abstract void onStop();
+  public abstract void onPostResume();
+  public abstract void setContentView(android.view.View);
+  public abstract void setContentView(int);
+  public abstract void setContentView(android.view.View, android.view.ViewGroup$LayoutParams);
+  public abstract void addContentView(android.view.View, android.view.ViewGroup$LayoutParams);
+  public abstract void setTitle(java.lang.CharSequence);
+  public abstract void invalidateOptionsMenu();
+  public abstract void onDestroy();
+  public abstract android.support.v7.app.ActionBarDrawerToggle$Delegate getDrawerToggleDelegate();
+  public abstract boolean requestWindowFeature(int);
+  public abstract android.support.v7.view.ActionMode startSupportActionMode(android.support.v7.view.ActionMode$Callback);
+  public abstract void installViewFactory();
+  public abstract android.view.View createView(android.view.View, java.lang.String, android.content.Context, android.util.AttributeSet);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatDialog appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatDialog
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.app.AppCompatDialog	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.app.AppCompatDialog	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,16 @@
+public class android.support.v7.app.AppCompatDialog extends android.app.Dialog implements android.support.v7.app.AppCompatCallback {
+  public android.support.v7.app.AppCompatDialog(android.content.Context);
+  public android.support.v7.app.AppCompatDialog(android.content.Context, int);
+  public android.support.v7.app.ActionBar getSupportActionBar();
+  public void setContentView(int);
+  public void setContentView(android.view.View);
+  public void setContentView(android.view.View, android.view.ViewGroup$LayoutParams);
+  public void setTitle(java.lang.CharSequence);
+  public void setTitle(int);
+  public void addContentView(android.view.View, android.view.ViewGroup$LayoutParams);
+  public boolean supportRequestWindowFeature(int);
+  public void invalidateOptionsMenu();
+  public android.support.v7.app.AppCompatDelegate getDelegate();
+  public void onSupportActionModeStarted(android.support.v7.view.ActionMode);
+  public void onSupportActionModeFinished(android.support.v7.view.ActionMode);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.graphics.drawable.DrawableWrapper appcompat-v7-22.1.0_0772a5ef/android.support.v7.graphics.drawable.DrawableWrapper
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.graphics.drawable.DrawableWrapper	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.graphics.drawable.DrawableWrapper	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,35 @@
+public class android.support.v7.graphics.drawable.DrawableWrapper extends android.graphics.drawable.Drawable implements android.graphics.drawable.Drawable$Callback {
+  public android.support.v7.graphics.drawable.DrawableWrapper(android.graphics.drawable.Drawable);
+  public void draw(android.graphics.Canvas);
+  public void setChangingConfigurations(int);
+  public int getChangingConfigurations();
+  public void setDither(boolean);
+  public void setFilterBitmap(boolean);
+  public void setAlpha(int);
+  public void setColorFilter(android.graphics.ColorFilter);
+  public boolean isStateful();
+  public boolean setState(int[]);
+  public int[] getState();
+  public void jumpToCurrentState();
+  public android.graphics.drawable.Drawable getCurrent();
+  public boolean setVisible(boolean, boolean);
+  public int getOpacity();
+  public android.graphics.Region getTransparentRegion();
+  public int getIntrinsicWidth();
+  public int getIntrinsicHeight();
+  public int getMinimumWidth();
+  public int getMinimumHeight();
+  public boolean getPadding(android.graphics.Rect);
+  public void invalidateDrawable(android.graphics.drawable.Drawable);
+  public void scheduleDrawable(android.graphics.drawable.Drawable, java.lang.Runnable, long);
+  public void unscheduleDrawable(android.graphics.drawable.Drawable, java.lang.Runnable);
+  public void setAutoMirrored(boolean);
+  public boolean isAutoMirrored();
+  public void setTint(int);
+  public void setTintList(android.content.res.ColorStateList);
+  public void setTintMode(android.graphics.PorterDuff$Mode);
+  public void setHotspot(float, float);
+  public void setHotspotBounds(int, int, int, int);
+  public android.graphics.drawable.Drawable getWrappedDrawable();
+  public void setWrappedDrawable(android.graphics.drawable.Drawable);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.AppCompatViewInflater appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.AppCompatViewInflater
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.AppCompatViewInflater	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.AppCompatViewInflater	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v7.internal.app.AppCompatViewInflater {
+  public android.support.v7.internal.app.AppCompatViewInflater(android.content.Context);
+  public final android.view.View createView(android.view.View, java.lang.String, android.content.Context, android.util.AttributeSet, boolean, boolean);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.ToolbarActionBar appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.ToolbarActionBar
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.ToolbarActionBar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.ToolbarActionBar	2015-08-20 23:41:44.000000000 +0900
@@ -2,2 +2,2 @@
-  public android.support.v7.internal.app.ToolbarActionBar(android.support.v7.widget.Toolbar, java.lang.CharSequence, android.view.Window, android.support.v7.internal.app.WindowCallback);
-  public android.support.v7.internal.app.WindowCallback getWrappedWindowCallback();
+  public android.support.v7.internal.app.ToolbarActionBar(android.support.v7.widget.Toolbar, java.lang.CharSequence, android.view.Window);
+  public android.view.Window$Callback getWrappedWindowCallback();
@@ -25 +24,0 @@
-  public android.support.v7.view.ActionMode startActionMode(android.support.v7.view.ActionMode$Callback);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.WindowCallback appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.WindowCallback
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.WindowCallback	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.WindowCallback	1970-01-01 09:00:00.000000000 +0900
@@ -1,9 +0,0 @@
-public interface android.support.v7.internal.app.WindowCallback {
-  public abstract boolean onMenuItemSelected(int, android.view.MenuItem);
-  public abstract boolean onCreatePanelMenu(int, android.view.Menu);
-  public abstract boolean onPreparePanel(int, android.view.View, android.view.Menu);
-  public abstract void onPanelClosed(int, android.view.Menu);
-  public abstract boolean onMenuOpened(int, android.view.Menu);
-  public abstract android.support.v7.view.ActionMode startActionMode(android.support.v7.view.ActionMode$Callback);
-  public abstract android.view.View onCreatePanelView(int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.WindowDecorActionBar appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.WindowDecorActionBar
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.app.WindowDecorActionBar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.app.WindowDecorActionBar	2015-08-20 23:41:44.000000000 +0900
@@ -2 +2 @@
-  public android.support.v7.internal.app.WindowDecorActionBar(android.support.v7.app.ActionBarActivity, boolean);
+  public android.support.v7.internal.app.WindowDecorActionBar(android.app.Activity, boolean);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.view.ContextThemeWrapper appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.view.ContextThemeWrapper
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.view.ContextThemeWrapper	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.view.ContextThemeWrapper	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,7 @@
+public class android.support.v7.internal.view.ContextThemeWrapper extends android.content.ContextWrapper {
+  public android.support.v7.internal.view.ContextThemeWrapper(android.content.Context, int);
+  public void setTheme(int);
+  public int getThemeResId();
+  public android.content.res.Resources$Theme getTheme();
+  public java.lang.Object getSystemService(java.lang.String);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.view.WindowCallbackWrapper appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.view.WindowCallbackWrapper
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.view.WindowCallbackWrapper	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.view.WindowCallbackWrapper	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,24 @@
+public class android.support.v7.internal.view.WindowCallbackWrapper implements android.view.Window$Callback {
+  public android.support.v7.internal.view.WindowCallbackWrapper(android.view.Window$Callback);
+  public boolean dispatchKeyEvent(android.view.KeyEvent);
+  public boolean dispatchKeyShortcutEvent(android.view.KeyEvent);
+  public boolean dispatchTouchEvent(android.view.MotionEvent);
+  public boolean dispatchTrackballEvent(android.view.MotionEvent);
+  public boolean dispatchGenericMotionEvent(android.view.MotionEvent);
+  public boolean dispatchPopulateAccessibilityEvent(android.view.accessibility.AccessibilityEvent);
+  public android.view.View onCreatePanelView(int);
+  public boolean onCreatePanelMenu(int, android.view.Menu);
+  public boolean onPreparePanel(int, android.view.View, android.view.Menu);
+  public boolean onMenuOpened(int, android.view.Menu);
+  public boolean onMenuItemSelected(int, android.view.MenuItem);
+  public void onWindowAttributesChanged(android.view.WindowManager$LayoutParams);
+  public void onContentChanged();
+  public void onWindowFocusChanged(boolean);
+  public void onAttachedToWindow();
+  public void onDetachedFromWindow();
+  public void onPanelClosed(int, android.view.Menu);
+  public boolean onSearchRequested();
+  public android.view.ActionMode onWindowStartingActionMode(android.view.ActionMode$Callback);
+  public void onActionModeStarted(android.view.ActionMode);
+  public void onActionModeFinished(android.view.ActionMode);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.view.menu.ActionMenuItemView appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.view.menu.ActionMenuItemView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.view.menu.ActionMenuItemView	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.view.menu.ActionMenuItemView	2015-08-20 23:41:44.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.internal.view.menu.ActionMenuItemView extends android.support.v7.internal.widget.CompatTextView implements android.support.v7.internal.view.menu.MenuView$ItemView,android.view.View$OnClickListener,android.view.View$OnLongClickListener,android.support.v7.widget.ActionMenuView$ActionMenuChildView {
+public class android.support.v7.internal.view.menu.ActionMenuItemView extends android.support.v7.widget.AppCompatTextView implements android.support.v7.internal.view.menu.MenuView$ItemView,android.view.View$OnClickListener,android.view.View$OnLongClickListener,android.support.v7.widget.ActionMenuView$ActionMenuChildView {
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ActionBarOverlayLayout appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ActionBarOverlayLayout
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ActionBarOverlayLayout	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ActionBarOverlayLayout	2015-08-20 23:41:44.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.internal.widget.ActionBarOverlayLayout extends android.view.ViewGroup implements android.support.v7.internal.widget.DecorContentParent {
+public class android.support.v7.internal.widget.ActionBarOverlayLayout extends android.view.ViewGroup implements android.support.v7.internal.widget.DecorContentParent,android.support.v4.view.NestedScrollingParent {
@@ -17,0 +18,3 @@
+  public void onNestedPreScroll(android.view.View, int, int, int[]);
+  public boolean onNestedPreFling(android.view.View, float, float);
+  public int getNestedScrollAxes();
@@ -22 +25 @@
-  public void setWindowCallback(android.support.v7.internal.app.WindowCallback);
+  public void setWindowCallback(android.view.Window$Callback);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.CompatTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.CompatTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.CompatTextView	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.CompatTextView	1970-01-01 09:00:00.000000000 +0900
@@ -1,5 +0,0 @@
-public class android.support.v7.internal.widget.CompatTextView extends android.widget.TextView {
-  public android.support.v7.internal.widget.CompatTextView(android.content.Context);
-  public android.support.v7.internal.widget.CompatTextView(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.CompatTextView(android.content.Context, android.util.AttributeSet, int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ContentFrameLayout appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ContentFrameLayout
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ContentFrameLayout	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ContentFrameLayout	2015-08-20 23:41:44.000000000 +0900
@@ -5,0 +6,7 @@
+  public void setDecorPadding(int, int, int, int);
+  public android.util.TypedValue getMinWidthMajor();
+  public android.util.TypedValue getMinWidthMinor();
+  public android.util.TypedValue getFixedWidthMajor();
+  public android.util.TypedValue getFixedWidthMinor();
+  public android.util.TypedValue getFixedHeightMajor();
+  public android.util.TypedValue getFixedHeightMinor();
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DecorContentParent appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DecorContentParent
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DecorContentParent	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DecorContentParent	2015-08-20 23:41:44.000000000 +0900
@@ -2 +2 @@
-  public abstract void setWindowCallback(android.support.v7.internal.app.WindowCallback);
+  public abstract void setWindowCallback(android.view.Window$Callback);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DecorToolbar appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DecorToolbar
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DecorToolbar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DecorToolbar	2015-08-20 23:41:44.000000000 +0900
@@ -7 +7 @@
-  public abstract void setWindowCallback(android.support.v7.internal.app.WindowCallback);
+  public abstract void setWindowCallback(android.view.Window$Callback);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DialogTitle appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DialogTitle
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DialogTitle	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DialogTitle	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v7.internal.widget.DialogTitle extends android.widget.TextView {
+  public android.support.v7.internal.widget.DialogTitle(android.content.Context, android.util.AttributeSet, int);
+  public android.support.v7.internal.widget.DialogTitle(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.internal.widget.DialogTitle(android.content.Context);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DrawableUtils appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DrawableUtils
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.DrawableUtils	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.DrawableUtils	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v7.internal.widget.DrawableUtils {
+  public static final android.graphics.Rect INSETS_NONE;
+  public static android.graphics.Rect getOpticalBounds(android.graphics.drawable.Drawable);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ThemeUtils appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ThemeUtils
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ThemeUtils	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ThemeUtils	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,7 @@
+public class android.support.v7.internal.widget.ThemeUtils {
+  public android.support.v7.internal.widget.ThemeUtils();
+  public static android.content.res.ColorStateList createDisabledStateList(int, int);
+  public static int getThemeAttrColor(android.content.Context, int);
+  public static android.content.res.ColorStateList getThemeAttrColorStateList(android.content.Context, int);
+  public static int getDisabledThemeAttrColor(android.content.Context, int);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintAutoCompleteTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintAutoCompleteTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintAutoCompleteTextView	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintAutoCompleteTextView	1970-01-01 09:00:00.000000000 +0900
@@ -1,6 +0,0 @@
-public class android.support.v7.internal.widget.TintAutoCompleteTextView extends android.widget.AutoCompleteTextView {
-  public android.support.v7.internal.widget.TintAutoCompleteTextView(android.content.Context);
-  public android.support.v7.internal.widget.TintAutoCompleteTextView(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintAutoCompleteTextView(android.content.Context, android.util.AttributeSet, int);
-  public void setDropDownBackgroundResource(int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintButton appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintButton
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintButton	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintButton	1970-01-01 09:00:00.000000000 +0900
@@ -1,6 +0,0 @@
-public class android.support.v7.internal.widget.TintButton extends android.widget.Button {
-  public android.support.v7.internal.widget.TintButton(android.content.Context);
-  public android.support.v7.internal.widget.TintButton(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintButton(android.content.Context, android.util.AttributeSet, int);
-  public void setBackgroundResource(int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintCheckBox appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintCheckBox
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintCheckBox	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintCheckBox	1970-01-01 09:00:00.000000000 +0900
@@ -1,6 +0,0 @@
-public class android.support.v7.internal.widget.TintCheckBox extends android.widget.CheckBox {
-  public android.support.v7.internal.widget.TintCheckBox(android.content.Context);
-  public android.support.v7.internal.widget.TintCheckBox(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintCheckBox(android.content.Context, android.util.AttributeSet, int);
-  public void setButtonDrawable(int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintCheckedTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintCheckedTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintCheckedTextView	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintCheckedTextView	1970-01-01 09:00:00.000000000 +0900
@@ -1,6 +0,0 @@
-public class android.support.v7.internal.widget.TintCheckedTextView extends android.widget.CheckedTextView {
-  public android.support.v7.internal.widget.TintCheckedTextView(android.content.Context);
-  public android.support.v7.internal.widget.TintCheckedTextView(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintCheckedTextView(android.content.Context, android.util.AttributeSet, int);
-  public void setCheckMarkDrawable(int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintContextWrapper appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintContextWrapper
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintContextWrapper	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintContextWrapper	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v7.internal.widget.TintContextWrapper extends android.content.ContextWrapper {
+  public static android.content.Context wrap(android.content.Context);
+  public android.content.res.Resources getResources();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintEditText appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintEditText
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintEditText	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintEditText	1970-01-01 09:00:00.000000000 +0900
@@ -1,5 +0,0 @@
-public class android.support.v7.internal.widget.TintEditText extends android.widget.EditText {
-  public android.support.v7.internal.widget.TintEditText(android.content.Context);
-  public android.support.v7.internal.widget.TintEditText(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintEditText(android.content.Context, android.util.AttributeSet, int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintInfo appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintInfo
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintInfo	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintInfo	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,7 @@
+public class android.support.v7.internal.widget.TintInfo {
+  public android.content.res.ColorStateList mTintList;
+  public android.graphics.PorterDuff$Mode mTintMode;
+  public boolean mHasTintMode;
+  public boolean mHasTintList;
+  public android.support.v7.internal.widget.TintInfo();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintManager appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintManager
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintManager	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintManager	2015-08-20 23:41:44.000000000 +0900
@@ -1 +1,2 @@
-public class android.support.v7.internal.widget.TintManager {
+public final class android.support.v7.internal.widget.TintManager {
+  public static final boolean SHOULD_BE_USED;
@@ -3 +4 @@
-  public android.support.v7.internal.widget.TintManager(android.content.Context);
+  public static android.support.v7.internal.widget.TintManager get(android.content.Context);
@@ -4,0 +6,4 @@
+  public android.graphics.drawable.Drawable getDrawable(int, boolean);
+  public final boolean tintDrawableUsingColorFilter(int, android.graphics.drawable.Drawable);
+  public final android.content.res.ColorStateList getTintList(int);
+  public static void tintViewBackground(android.view.View, android.support.v7.internal.widget.TintInfo);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintMultiAutoCompleteTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintMultiAutoCompleteTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintMultiAutoCompleteTextView	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintMultiAutoCompleteTextView	1970-01-01 09:00:00.000000000 +0900
@@ -1,6 +0,0 @@
-public class android.support.v7.internal.widget.TintMultiAutoCompleteTextView extends android.widget.MultiAutoCompleteTextView {
-  public android.support.v7.internal.widget.TintMultiAutoCompleteTextView(android.content.Context);
-  public android.support.v7.internal.widget.TintMultiAutoCompleteTextView(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintMultiAutoCompleteTextView(android.content.Context, android.util.AttributeSet, int);
-  public void setDropDownBackgroundResource(int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintRadioButton appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintRadioButton
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintRadioButton	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintRadioButton	1970-01-01 09:00:00.000000000 +0900
@@ -1,6 +0,0 @@
-public class android.support.v7.internal.widget.TintRadioButton extends android.widget.RadioButton {
-  public android.support.v7.internal.widget.TintRadioButton(android.content.Context);
-  public android.support.v7.internal.widget.TintRadioButton(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintRadioButton(android.content.Context, android.util.AttributeSet, int);
-  public void setButtonDrawable(int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintRatingBar appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintRatingBar
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintRatingBar	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintRatingBar	1970-01-01 09:00:00.000000000 +0900
@@ -1,5 +0,0 @@
-public class android.support.v7.internal.widget.TintRatingBar extends android.widget.RatingBar {
-  public android.support.v7.internal.widget.TintRatingBar(android.content.Context);
-  public android.support.v7.internal.widget.TintRatingBar(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintRatingBar(android.content.Context, android.util.AttributeSet, int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintSpinner appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintSpinner
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintSpinner	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintSpinner	1970-01-01 09:00:00.000000000 +0900
@@ -1,5 +0,0 @@
-public class android.support.v7.internal.widget.TintSpinner extends android.widget.Spinner {
-  public android.support.v7.internal.widget.TintSpinner(android.content.Context);
-  public android.support.v7.internal.widget.TintSpinner(android.content.Context, android.util.AttributeSet);
-  public android.support.v7.internal.widget.TintSpinner(android.content.Context, android.util.AttributeSet, int);
-}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintTypedArray appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintTypedArray
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.TintTypedArray	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.TintTypedArray	2015-08-20 23:41:44.000000000 +0900
@@ -4,0 +5 @@
+  public android.graphics.drawable.Drawable getDrawableIfKnown(int);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ToolbarWidgetWrapper appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ToolbarWidgetWrapper
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ToolbarWidgetWrapper	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ToolbarWidgetWrapper	2015-08-20 23:41:44.000000000 +0900
@@ -11 +11 @@
-  public void setWindowCallback(android.support.v7.internal.app.WindowCallback);
+  public void setWindowCallback(android.view.Window$Callback);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ViewUtils appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ViewUtils
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.internal.widget.ViewUtils	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.internal.widget.ViewUtils	2015-08-20 23:41:44.000000000 +0900
@@ -5,0 +6 @@
+  public static android.content.Context themifyContext(android.content.Context, android.util.AttributeSet, boolean, boolean);
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatAutoCompleteTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatAutoCompleteTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatAutoCompleteTextView	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatAutoCompleteTextView	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,10 @@
+public class android.support.v7.widget.AppCompatAutoCompleteTextView extends android.widget.AutoCompleteTextView implements android.support.v4.view.TintableBackgroundView {
+  public android.support.v7.widget.AppCompatAutoCompleteTextView(android.content.Context);
+  public android.support.v7.widget.AppCompatAutoCompleteTextView(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatAutoCompleteTextView(android.content.Context, android.util.AttributeSet, int);
+  public void setDropDownBackgroundResource(int);
+  public void setSupportBackgroundTintList(android.content.res.ColorStateList);
+  public android.content.res.ColorStateList getSupportBackgroundTintList();
+  public void setSupportBackgroundTintMode(android.graphics.PorterDuff$Mode);
+  public android.graphics.PorterDuff$Mode getSupportBackgroundTintMode();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatButton appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatButton
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatButton	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatButton	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,13 @@
+public class android.support.v7.widget.AppCompatButton extends android.widget.Button implements android.support.v4.view.TintableBackgroundView {
+  public android.support.v7.widget.AppCompatButton(android.content.Context);
+  public android.support.v7.widget.AppCompatButton(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatButton(android.content.Context, android.util.AttributeSet, int);
+  public void setSupportBackgroundTintList(android.content.res.ColorStateList);
+  public android.content.res.ColorStateList getSupportBackgroundTintList();
+  public void setSupportBackgroundTintMode(android.graphics.PorterDuff$Mode);
+  public android.graphics.PorterDuff$Mode getSupportBackgroundTintMode();
+  public void onInitializeAccessibilityEvent(android.view.accessibility.AccessibilityEvent);
+  public void onInitializeAccessibilityNodeInfo(android.view.accessibility.AccessibilityNodeInfo);
+  public void setAllCaps(boolean);
+  public void setTextAppearance(android.content.Context, int);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatCheckBox appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatCheckBox
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatCheckBox	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatCheckBox	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,8 @@
+public class android.support.v7.widget.AppCompatCheckBox extends android.widget.CheckBox {
+  public android.support.v7.widget.AppCompatCheckBox(android.content.Context);
+  public android.support.v7.widget.AppCompatCheckBox(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatCheckBox(android.content.Context, android.util.AttributeSet, int);
+  public void setButtonDrawable(android.graphics.drawable.Drawable);
+  public void setButtonDrawable(int);
+  public int getCompoundPaddingLeft();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatCheckedTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatCheckedTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatCheckedTextView	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatCheckedTextView	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v7.widget.AppCompatCheckedTextView extends android.widget.CheckedTextView {
+  public android.support.v7.widget.AppCompatCheckedTextView(android.content.Context);
+  public android.support.v7.widget.AppCompatCheckedTextView(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatCheckedTextView(android.content.Context, android.util.AttributeSet, int);
+  public void setCheckMarkDrawable(int);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatEditText appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatEditText
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatEditText	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatEditText	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,9 @@
+public class android.support.v7.widget.AppCompatEditText extends android.widget.EditText implements android.support.v4.view.TintableBackgroundView {
+  public android.support.v7.widget.AppCompatEditText(android.content.Context);
+  public android.support.v7.widget.AppCompatEditText(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatEditText(android.content.Context, android.util.AttributeSet, int);
+  public void setSupportBackgroundTintList(android.content.res.ColorStateList);
+  public android.content.res.ColorStateList getSupportBackgroundTintList();
+  public void setSupportBackgroundTintMode(android.graphics.PorterDuff$Mode);
+  public android.graphics.PorterDuff$Mode getSupportBackgroundTintMode();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatMultiAutoCompleteTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatMultiAutoCompleteTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatMultiAutoCompleteTextView	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatMultiAutoCompleteTextView	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,10 @@
+public class android.support.v7.widget.AppCompatMultiAutoCompleteTextView extends android.widget.MultiAutoCompleteTextView implements android.support.v4.view.TintableBackgroundView {
+  public android.support.v7.widget.AppCompatMultiAutoCompleteTextView(android.content.Context);
+  public android.support.v7.widget.AppCompatMultiAutoCompleteTextView(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatMultiAutoCompleteTextView(android.content.Context, android.util.AttributeSet, int);
+  public void setDropDownBackgroundResource(int);
+  public void setSupportBackgroundTintList(android.content.res.ColorStateList);
+  public android.content.res.ColorStateList getSupportBackgroundTintList();
+  public void setSupportBackgroundTintMode(android.graphics.PorterDuff$Mode);
+  public android.graphics.PorterDuff$Mode getSupportBackgroundTintMode();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatRadioButton appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatRadioButton
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatRadioButton	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatRadioButton	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,8 @@
+public class android.support.v7.widget.AppCompatRadioButton extends android.widget.RadioButton {
+  public android.support.v7.widget.AppCompatRadioButton(android.content.Context);
+  public android.support.v7.widget.AppCompatRadioButton(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatRadioButton(android.content.Context, android.util.AttributeSet, int);
+  public void setButtonDrawable(android.graphics.drawable.Drawable);
+  public void setButtonDrawable(int);
+  public int getCompoundPaddingLeft();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatRatingBar appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatRatingBar
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatRatingBar	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatRatingBar	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v7.widget.AppCompatRatingBar extends android.widget.RatingBar {
+  public android.support.v7.widget.AppCompatRatingBar(android.content.Context);
+  public android.support.v7.widget.AppCompatRatingBar(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatRatingBar(android.content.Context, android.util.AttributeSet, int);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatSpinner appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatSpinner
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatSpinner	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatSpinner	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,9 @@
+public class android.support.v7.widget.AppCompatSpinner extends android.widget.Spinner implements android.support.v4.view.TintableBackgroundView {
+  public android.support.v7.widget.AppCompatSpinner(android.content.Context);
+  public android.support.v7.widget.AppCompatSpinner(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatSpinner(android.content.Context, android.util.AttributeSet, int);
+  public void setSupportBackgroundTintList(android.content.res.ColorStateList);
+  public android.content.res.ColorStateList getSupportBackgroundTintList();
+  public void setSupportBackgroundTintMode(android.graphics.PorterDuff$Mode);
+  public android.graphics.PorterDuff$Mode getSupportBackgroundTintMode();
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatTextView appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatTextView
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.AppCompatTextView	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.AppCompatTextView	2015-08-20 23:41:44.000000000 +0900
@@ -0,0 +1,7 @@
+public class android.support.v7.widget.AppCompatTextView extends android.widget.TextView {
+  public android.support.v7.widget.AppCompatTextView(android.content.Context);
+  public android.support.v7.widget.AppCompatTextView(android.content.Context, android.util.AttributeSet);
+  public android.support.v7.widget.AppCompatTextView(android.content.Context, android.util.AttributeSet, int);
+  public void setAllCaps(boolean);
+  public void setTextAppearance(android.content.Context, int);
+}
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.SearchView$SearchAutoComplete appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.SearchView$SearchAutoComplete
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.SearchView$SearchAutoComplete	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.SearchView$SearchAutoComplete	2015-08-20 23:41:44.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.widget.SearchView$SearchAutoComplete extends android.support.v7.internal.widget.TintAutoCompleteTextView {
+public class android.support.v7.widget.SearchView$SearchAutoComplete extends android.support.v7.widget.AppCompatAutoCompleteTextView {
diff -U 0 -N appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.WindowCallbackWrapper appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.WindowCallbackWrapper
--- appcompat-v7-22.0.0_6bfba991/android.support.v7.widget.WindowCallbackWrapper	2015-08-20 23:41:44.000000000 +0900
+++ appcompat-v7-22.1.0_0772a5ef/android.support.v7.widget.WindowCallbackWrapper	1970-01-01 09:00:00.000000000 +0900
@@ -1,10 +0,0 @@
-public class android.support.v7.widget.WindowCallbackWrapper implements android.support.v7.internal.app.WindowCallback {
-  public android.support.v7.widget.WindowCallbackWrapper(android.support.v7.internal.app.WindowCallback);
-  public boolean onMenuItemSelected(int, android.view.MenuItem);
-  public boolean onCreatePanelMenu(int, android.view.Menu);
-  public boolean onPreparePanel(int, android.view.View, android.view.Menu);
-  public void onPanelClosed(int, android.view.Menu);
-  public boolean onMenuOpened(int, android.view.Menu);
-  public android.support.v7.view.ActionMode startActionMode(android.support.v7.view.ActionMode$Callback);
-  public android.view.View onCreatePanelView(int);
-}
