diff -U 0 -N appcompat-v7-23.2.0_93bf5bc2/android.support.v7.app.AppCompatActivity appcompat-v7-23.2.1_a5c5d54a/android.support.v7.app.AppCompatActivity
--- appcompat-v7-23.2.0_93bf5bc2/android.support.v7.app.AppCompatActivity	2016-03-10 19:56:36.000000000 +0900
+++ appcompat-v7-23.2.1_a5c5d54a/android.support.v7.app.AppCompatActivity	2016-03-10 19:57:14.000000000 +0900
@@ -11,0 +12 @@
+  public android.view.View findViewById(int);
diff -U 0 -N appcompat-v7-23.2.0_93bf5bc2/android.support.v7.app.AppCompatDelegate appcompat-v7-23.2.1_a5c5d54a/android.support.v7.app.AppCompatDelegate
--- appcompat-v7-23.2.0_93bf5bc2/android.support.v7.app.AppCompatDelegate	2016-03-10 19:56:36.000000000 +0900
+++ appcompat-v7-23.2.1_a5c5d54a/android.support.v7.app.AppCompatDelegate	2016-03-10 19:57:14.000000000 +0900
@@ -18,0 +19 @@
+  public abstract android.view.View findViewById(int);
diff -U 0 -N appcompat-v7-23.2.0_93bf5bc2/android.support.v7.app.AppCompatDialog appcompat-v7-23.2.1_a5c5d54a/android.support.v7.app.AppCompatDialog
--- appcompat-v7-23.2.0_93bf5bc2/android.support.v7.app.AppCompatDialog	2016-03-10 19:56:36.000000000 +0900
+++ appcompat-v7-23.2.1_a5c5d54a/android.support.v7.app.AppCompatDialog	2016-03-10 19:57:14.000000000 +0900
@@ -7,0 +8 @@
+  public android.view.View findViewById(int);
diff -U 0 -N appcompat-v7-23.2.0_93bf5bc2/android.support.v7.widget.AppCompatImageHelper appcompat-v7-23.2.1_a5c5d54a/android.support.v7.widget.AppCompatImageHelper
--- appcompat-v7-23.2.0_93bf5bc2/android.support.v7.widget.AppCompatImageHelper	1970-01-01 09:00:00.000000000 +0900
+++ appcompat-v7-23.2.1_a5c5d54a/android.support.v7.widget.AppCompatImageHelper	2016-03-10 19:57:14.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v7.widget.AppCompatImageHelper {
+  public android.support.v7.widget.AppCompatImageHelper(android.widget.ImageView, android.support.v7.widget.AppCompatDrawableManager);
+  public void loadFromAttributes(android.util.AttributeSet, int);
+  public void setImageResource(int);
+}
