diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.BuildConfig support-v4-22.0.0_96dba0e2/android.support.v4.BuildConfig
--- support-v4-21.0.3_998b88de/android.support.v4.BuildConfig	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.BuildConfig	2015-09-05 18:52:10.000000000 +0900
@@ -3 +3 @@
-  public static final java.lang.String PACKAGE_NAME;
+  public static final java.lang.String APPLICATION_ID;
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation support-v4-22.0.0_96dba0e2/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation
--- support-v4-21.0.3_998b88de/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation	2015-09-05 18:52:10.000000000 +0900
@@ -1,0 +2,8 @@
+  public java.lang.String[] getMessages();
+  public android.support.v4.app.RemoteInput getRemoteInput();
+  public android.app.PendingIntent getReplyPendingIntent();
+  public android.app.PendingIntent getReadPendingIntent();
+  public java.lang.String[] getParticipants();
+  public java.lang.String getParticipant();
+  public long getLatestTimestamp();
+  public android.support.v4.app.RemoteInputCompatBase$RemoteInput getRemoteInput();
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.content.res.ResourcesCompat support-v4-22.0.0_96dba0e2/android.support.v4.content.res.ResourcesCompat
--- support-v4-21.0.3_998b88de/android.support.v4.content.res.ResourcesCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.content.res.ResourcesCompat	2015-09-05 18:52:10.000000000 +0900
@@ -3,0 +4 @@
+  public static android.graphics.drawable.Drawable getDrawableForDensity(android.content.res.Resources, int, int, android.content.res.Resources$Theme) throws android.content.res.Resources$NotFoundException;
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.MediaDescriptionCompat support-v4-22.0.0_96dba0e2/android.support.v4.media.MediaDescriptionCompat
--- support-v4-21.0.3_998b88de/android.support.v4.media.MediaDescriptionCompat	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.MediaDescriptionCompat	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,15 @@
+public final class android.support.v4.media.MediaDescriptionCompat implements android.os.Parcelable {
+  public static final android.os.Parcelable$Creator<android.support.v4.media.MediaDescriptionCompat> CREATOR;
+  public java.lang.String getMediaId();
+  public java.lang.CharSequence getTitle();
+  public java.lang.CharSequence getSubtitle();
+  public java.lang.CharSequence getDescription();
+  public android.graphics.Bitmap getIconBitmap();
+  public android.net.Uri getIconUri();
+  public android.os.Bundle getExtras();
+  public int describeContents();
+  public void writeToParcel(android.os.Parcel, int);
+  public java.lang.String toString();
+  public java.lang.Object getMediaDescription();
+  public static android.support.v4.media.MediaDescriptionCompat fromMediaDescription(java.lang.Object);
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.MediaDescriptionCompat$Builder support-v4-22.0.0_96dba0e2/android.support.v4.media.MediaDescriptionCompat$Builder
--- support-v4-21.0.3_998b88de/android.support.v4.media.MediaDescriptionCompat$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.MediaDescriptionCompat$Builder	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,11 @@
+public final class android.support.v4.media.MediaDescriptionCompat$Builder {
+  public android.support.v4.media.MediaDescriptionCompat$Builder();
+  public android.support.v4.media.MediaDescriptionCompat$Builder setMediaId(java.lang.String);
+  public android.support.v4.media.MediaDescriptionCompat$Builder setTitle(java.lang.CharSequence);
+  public android.support.v4.media.MediaDescriptionCompat$Builder setSubtitle(java.lang.CharSequence);
+  public android.support.v4.media.MediaDescriptionCompat$Builder setDescription(java.lang.CharSequence);
+  public android.support.v4.media.MediaDescriptionCompat$Builder setIconBitmap(android.graphics.Bitmap);
+  public android.support.v4.media.MediaDescriptionCompat$Builder setIconUri(android.net.Uri);
+  public android.support.v4.media.MediaDescriptionCompat$Builder setExtras(android.os.Bundle);
+  public android.support.v4.media.MediaDescriptionCompat build();
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.MediaMetadataCompat support-v4-22.0.0_96dba0e2/android.support.v4.media.MediaMetadataCompat
--- support-v4-21.0.3_998b88de/android.support.v4.media.MediaMetadataCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.MediaMetadataCompat	2015-09-05 18:52:10.000000000 +0900
@@ -27,0 +28 @@
+  public static final java.lang.String METADATA_KEY_MEDIA_ID;
@@ -34,0 +36 @@
+  public android.support.v4.media.MediaDescriptionCompat getDescription();
@@ -38,0 +41 @@
+  public android.os.Bundle getBundle();
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaControllerCallback support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaControllerCallback
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaControllerCallback	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaControllerCallback	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,10 @@
+public interface android.support.v4.media.session.IMediaControllerCallback extends android.os.IInterface {
+  public abstract void onEvent(java.lang.String, android.os.Bundle) throws android.os.RemoteException;
+  public abstract void onSessionDestroyed() throws android.os.RemoteException;
+  public abstract void onPlaybackStateChanged(android.support.v4.media.session.PlaybackStateCompat) throws android.os.RemoteException;
+  public abstract void onMetadataChanged(android.support.v4.media.MediaMetadataCompat) throws android.os.RemoteException;
+  public abstract void onQueueChanged(java.util.List<android.support.v4.media.session.MediaSessionCompat$QueueItem>) throws android.os.RemoteException;
+  public abstract void onQueueTitleChanged(java.lang.CharSequence) throws android.os.RemoteException;
+  public abstract void onExtrasChanged(android.os.Bundle) throws android.os.RemoteException;
+  public abstract void onVolumeInfoChanged(android.support.v4.media.session.ParcelableVolumeInfo) throws android.os.RemoteException;
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaControllerCallback$Stub support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaControllerCallback$Stub
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaControllerCallback$Stub	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaControllerCallback$Stub	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,6 @@
+public abstract class android.support.v4.media.session.IMediaControllerCallback$Stub extends android.os.Binder implements android.support.v4.media.session.IMediaControllerCallback {
+  public android.support.v4.media.session.IMediaControllerCallback$Stub();
+  public static android.support.v4.media.session.IMediaControllerCallback asInterface(android.os.IBinder);
+  public android.os.IBinder asBinder();
+  public boolean onTransact(int, android.os.Parcel, android.os.Parcel, int) throws android.os.RemoteException;
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaSession support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaSession
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaSession	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaSession	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,33 @@
+public interface android.support.v4.media.session.IMediaSession extends android.os.IInterface {
+  public abstract void sendCommand(java.lang.String, android.os.Bundle, android.support.v4.media.session.MediaSessionCompat$ResultReceiverWrapper) throws android.os.RemoteException;
+  public abstract boolean sendMediaButton(android.view.KeyEvent) throws android.os.RemoteException;
+  public abstract void registerCallbackListener(android.support.v4.media.session.IMediaControllerCallback) throws android.os.RemoteException;
+  public abstract void unregisterCallbackListener(android.support.v4.media.session.IMediaControllerCallback) throws android.os.RemoteException;
+  public abstract boolean isTransportControlEnabled() throws android.os.RemoteException;
+  public abstract java.lang.String getPackageName() throws android.os.RemoteException;
+  public abstract java.lang.String getTag() throws android.os.RemoteException;
+  public abstract android.app.PendingIntent getLaunchPendingIntent() throws android.os.RemoteException;
+  public abstract long getFlags() throws android.os.RemoteException;
+  public abstract android.support.v4.media.session.ParcelableVolumeInfo getVolumeAttributes() throws android.os.RemoteException;
+  public abstract void adjustVolume(int, int, java.lang.String) throws android.os.RemoteException;
+  public abstract void setVolumeTo(int, int, java.lang.String) throws android.os.RemoteException;
+  public abstract void play() throws android.os.RemoteException;
+  public abstract void playFromMediaId(java.lang.String, android.os.Bundle) throws android.os.RemoteException;
+  public abstract void playFromSearch(java.lang.String, android.os.Bundle) throws android.os.RemoteException;
+  public abstract void skipToQueueItem(long) throws android.os.RemoteException;
+  public abstract void pause() throws android.os.RemoteException;
+  public abstract void stop() throws android.os.RemoteException;
+  public abstract void next() throws android.os.RemoteException;
+  public abstract void previous() throws android.os.RemoteException;
+  public abstract void fastForward() throws android.os.RemoteException;
+  public abstract void rewind() throws android.os.RemoteException;
+  public abstract void seekTo(long) throws android.os.RemoteException;
+  public abstract void rate(android.support.v4.media.RatingCompat) throws android.os.RemoteException;
+  public abstract void sendCustomAction(java.lang.String, android.os.Bundle) throws android.os.RemoteException;
+  public abstract android.support.v4.media.MediaMetadataCompat getMetadata() throws android.os.RemoteException;
+  public abstract android.support.v4.media.session.PlaybackStateCompat getPlaybackState() throws android.os.RemoteException;
+  public abstract java.util.List<android.support.v4.media.session.MediaSessionCompat$QueueItem> getQueue() throws android.os.RemoteException;
+  public abstract java.lang.CharSequence getQueueTitle() throws android.os.RemoteException;
+  public abstract android.os.Bundle getExtras() throws android.os.RemoteException;
+  public abstract int getRatingType() throws android.os.RemoteException;
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaSession$Stub support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaSession$Stub
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.IMediaSession$Stub	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.IMediaSession$Stub	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,6 @@
+public abstract class android.support.v4.media.session.IMediaSession$Stub extends android.os.Binder implements android.support.v4.media.session.IMediaSession {
+  public android.support.v4.media.session.IMediaSession$Stub();
+  public static android.support.v4.media.session.IMediaSession asInterface(android.os.IBinder);
+  public android.os.IBinder asBinder();
+  public boolean onTransact(int, android.os.Parcel, android.os.Parcel, int) throws android.os.RemoteException;
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaControllerCompat support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaControllerCompat
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaControllerCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaControllerCompat	2015-09-05 18:52:10.000000000 +0900
@@ -7,0 +8,3 @@
+  public java.util.List<android.support.v4.media.session.MediaSessionCompat$QueueItem> getQueue();
+  public java.lang.CharSequence getQueueTitle();
+  public android.os.Bundle getExtras();
@@ -8,0 +12 @@
+  public long getFlags();
@@ -9,0 +14,4 @@
+  public android.app.PendingIntent getSessionActivity();
+  public android.support.v4.media.session.MediaSessionCompat$Token getSessionToken();
+  public void setVolumeTo(int, int);
+  public void adjustVolume(int, int);
@@ -13,0 +22 @@
+  public java.lang.String getPackageName();
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaControllerCompat$Callback support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaControllerCompat$Callback
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaControllerCompat$Callback	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaControllerCompat$Callback	2015-09-05 18:52:10.000000000 +0900
@@ -1 +1 @@
-public abstract class android.support.v4.media.session.MediaControllerCompat$Callback {
+public abstract class android.support.v4.media.session.MediaControllerCompat$Callback implements android.os.IBinder$DeathRecipient {
@@ -6,0 +7,5 @@
+  public void onQueueChanged(java.util.List<android.support.v4.media.session.MediaSessionCompat$QueueItem>);
+  public void onQueueTitleChanged(java.lang.CharSequence);
+  public void onExtrasChanged(android.os.Bundle);
+  public void onAudioInfoChanged(android.support.v4.media.session.MediaControllerCompat$PlaybackInfo);
+  public void binderDied();
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaControllerCompat$TransportControls support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaControllerCompat$TransportControls
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaControllerCompat$TransportControls	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaControllerCompat$TransportControls	2015-09-05 18:52:10.000000000 +0900
@@ -2,0 +3,3 @@
+  public abstract void playFromMediaId(java.lang.String, android.os.Bundle);
+  public abstract void playFromSearch(java.lang.String, android.os.Bundle);
+  public abstract void skipToQueueItem(long);
@@ -10,0 +14,2 @@
+  public abstract void sendCustomAction(android.support.v4.media.session.PlaybackStateCompat$CustomAction, android.os.Bundle);
+  public abstract void sendCustomAction(java.lang.String, android.os.Bundle);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat	2015-09-05 18:52:10.000000000 +0900
@@ -4 +4 @@
-  public android.support.v4.media.session.MediaSessionCompat(android.content.Context, java.lang.String);
+  public android.support.v4.media.session.MediaSessionCompat(android.content.Context, java.lang.String, android.content.ComponentName, android.app.PendingIntent);
@@ -6,0 +7,2 @@
+  public void setSessionActivity(android.app.PendingIntent);
+  public void setMediaButtonReceiver(android.app.PendingIntent);
@@ -14,0 +17 @@
+  public android.support.v4.media.session.MediaControllerCompat getController();
@@ -16,0 +20,4 @@
+  public void setQueue(java.util.List<android.support.v4.media.session.MediaSessionCompat$QueueItem>);
+  public void setQueueTitle(java.lang.CharSequence);
+  public void setRatingType(int);
+  public void setExtras(android.os.Bundle);
@@ -18 +25,4 @@
-  public static android.support.v4.media.session.MediaSessionCompat obtain(java.lang.Object);
+  public java.lang.Object getRemoteControlClient();
+  public void addOnActiveChangeListener(android.support.v4.media.session.MediaSessionCompat$OnActiveChangeListener);
+  public void removeOnActiveChangeListener(android.support.v4.media.session.MediaSessionCompat$OnActiveChangeListener);
+  public static android.support.v4.media.session.MediaSessionCompat obtain(android.content.Context, java.lang.Object);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$Callback support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$Callback
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$Callback	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$Callback	2015-09-05 18:52:10.000000000 +0900
@@ -5,0 +6,3 @@
+  public void onPlayFromMediaId(java.lang.String, android.os.Bundle);
+  public void onPlayFromSearch(java.lang.String, android.os.Bundle);
+  public void onSkipToQueueItem(long);
@@ -13,0 +17 @@
+  public void onCustomAction(java.lang.String, android.os.Bundle);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$OnActiveChangeListener support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$OnActiveChangeListener
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$OnActiveChangeListener	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$OnActiveChangeListener	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.media.session.MediaSessionCompat$OnActiveChangeListener {
+  public abstract void onActiveChanged();
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$QueueItem support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$QueueItem
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$QueueItem	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$QueueItem	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,12 @@
+public final class android.support.v4.media.session.MediaSessionCompat$QueueItem implements android.os.Parcelable {
+  public static final int UNKNOWN_ID;
+  public static final android.os.Parcelable$Creator<android.support.v4.media.session.MediaSessionCompat$QueueItem> CREATOR;
+  public android.support.v4.media.session.MediaSessionCompat$QueueItem(android.support.v4.media.MediaDescriptionCompat, long);
+  public android.support.v4.media.MediaDescriptionCompat getDescription();
+  public long getQueueId();
+  public void writeToParcel(android.os.Parcel, int);
+  public int describeContents();
+  public java.lang.Object getQueueItem();
+  public static android.support.v4.media.session.MediaSessionCompat$QueueItem obtain(java.lang.Object);
+  public java.lang.String toString();
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$Token support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$Token
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.MediaSessionCompat$Token	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.MediaSessionCompat$Token	2015-09-05 18:52:10.000000000 +0900
@@ -2,0 +3 @@
+  public static android.support.v4.media.session.MediaSessionCompat$Token fromToken(java.lang.Object);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.ParcelableVolumeInfo support-v4-22.0.0_96dba0e2/android.support.v4.media.session.ParcelableVolumeInfo
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.ParcelableVolumeInfo	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.ParcelableVolumeInfo	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,12 @@
+public class android.support.v4.media.session.ParcelableVolumeInfo implements android.os.Parcelable {
+  public int volumeType;
+  public int audioStream;
+  public int controlType;
+  public int maxVolume;
+  public int currentVolume;
+  public static final android.os.Parcelable$Creator<android.support.v4.media.session.ParcelableVolumeInfo> CREATOR;
+  public android.support.v4.media.session.ParcelableVolumeInfo(int, int, int, int, int);
+  public android.support.v4.media.session.ParcelableVolumeInfo(android.os.Parcel);
+  public int describeContents();
+  public void writeToParcel(android.os.Parcel, int);
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.PlaybackStateCompat$Builder support-v4-22.0.0_96dba0e2/android.support.v4.media.session.PlaybackStateCompat$Builder
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.PlaybackStateCompat$Builder	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.PlaybackStateCompat$Builder	2015-09-05 18:52:10.000000000 +0900
@@ -4,4 +4,5 @@
-  public void setState(int, long, float);
-  public void setBufferedPosition(long);
-  public void setActions(long);
-  public void setErrorMessage(java.lang.CharSequence);
+  public android.support.v4.media.session.PlaybackStateCompat$Builder setState(int, long, float);
+  public android.support.v4.media.session.PlaybackStateCompat$Builder setState(int, long, float, long);
+  public android.support.v4.media.session.PlaybackStateCompat$Builder setBufferedPosition(long);
+  public android.support.v4.media.session.PlaybackStateCompat$Builder setActions(long);
+  public android.support.v4.media.session.PlaybackStateCompat$Builder setErrorMessage(java.lang.CharSequence);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.PlaybackStateCompat$CustomAction support-v4-22.0.0_96dba0e2/android.support.v4.media.session.PlaybackStateCompat$CustomAction
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.PlaybackStateCompat$CustomAction	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.PlaybackStateCompat$CustomAction	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,10 @@
+public final class android.support.v4.media.session.PlaybackStateCompat$CustomAction implements android.os.Parcelable {
+  public static final android.os.Parcelable$Creator<android.support.v4.media.session.PlaybackStateCompat$CustomAction> CREATOR;
+  public void writeToParcel(android.os.Parcel, int);
+  public int describeContents();
+  public java.lang.String getAction();
+  public java.lang.CharSequence getName();
+  public int getIcon();
+  public android.os.Bundle getExtras();
+  public java.lang.String toString();
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.media.session.PlaybackStateCompat$CustomAction$Builder support-v4-22.0.0_96dba0e2/android.support.v4.media.session.PlaybackStateCompat$CustomAction$Builder
--- support-v4-21.0.3_998b88de/android.support.v4.media.session.PlaybackStateCompat$CustomAction$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.media.session.PlaybackStateCompat$CustomAction$Builder	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,5 @@
+public final class android.support.v4.media.session.PlaybackStateCompat$CustomAction$Builder {
+  public android.support.v4.media.session.PlaybackStateCompat$CustomAction$Builder(java.lang.String, java.lang.CharSequence, int);
+  public android.support.v4.media.session.PlaybackStateCompat$CustomAction$Builder setExtras(android.os.Bundle);
+  public android.support.v4.media.session.PlaybackStateCompat$CustomAction build();
+}
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.view.ViewCompat support-v4-22.0.0_96dba0e2/android.support.v4.view.ViewCompat
--- support-v4-21.0.3_998b88de/android.support.v4.view.ViewCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.view.ViewCompat	2015-09-05 18:52:10.000000000 +0900
@@ -99,0 +100 @@
+  public static void setFitsSystemWindows(android.view.View, boolean);
@@ -101,0 +103,2 @@
+  public static void setSaveFromParentEnabled(android.view.View, boolean);
+  public static void setActivated(android.view.View, boolean);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat support-v4-22.0.0_96dba0e2/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat
--- support-v4-21.0.3_998b88de/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat	2015-09-05 18:52:10.000000000 +0900
@@ -50,0 +51 @@
+  public void addAction(android.support.v4.view.accessibility.AccessibilityNodeInfoCompat$AccessibilityActionCompat);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat$AccessibilityActionCompat support-v4-22.0.0_96dba0e2/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat$AccessibilityActionCompat
--- support-v4-21.0.3_998b88de/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat$AccessibilityActionCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.view.accessibility.AccessibilityNodeInfoCompat$AccessibilityActionCompat	2015-09-05 18:52:10.000000000 +0900
@@ -1,0 +2 @@
+  public android.support.v4.view.accessibility.AccessibilityNodeInfoCompat$AccessibilityActionCompat(int, java.lang.CharSequence);
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.widget.DrawerLayout support-v4-22.0.0_96dba0e2/android.support.v4.widget.DrawerLayout
--- support-v4-21.0.3_998b88de/android.support.v4.widget.DrawerLayout	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.widget.DrawerLayout	2015-09-05 18:52:10.000000000 +0900
@@ -25,0 +26 @@
+  public android.graphics.drawable.Drawable getStatusBarBackgroundDrawable();
diff -U 0 -N support-v4-21.0.3_998b88de/android.support.v4.widget.SwipeRefreshLayout support-v4-22.0.0_96dba0e2/android.support.v4.widget.SwipeRefreshLayout
--- support-v4-21.0.3_998b88de/android.support.v4.widget.SwipeRefreshLayout	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-22.0.0_96dba0e2/android.support.v4.widget.SwipeRefreshLayout	2015-09-05 18:52:10.000000000 +0900
@@ -11,0 +12,2 @@
+  public void setProgressBackgroundColorSchemeResource(int);
+  public void setProgressBackgroundColorSchemeColor(int);
@@ -17,0 +20 @@
+  public int getProgressCircleDiameter();
