diff -U 0 -N support-v4-21.0.0_55ef455a/android.support.v4.app.NotificationCompat$CarExtender support-v4-21.0.2_6496e6a2/android.support.v4.app.NotificationCompat$CarExtender
--- support-v4-21.0.0_55ef455a/android.support.v4.app.NotificationCompat$CarExtender	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-21.0.2_6496e6a2/android.support.v4.app.NotificationCompat$CarExtender	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,11 @@
+public final class android.support.v4.app.NotificationCompat$CarExtender implements android.support.v4.app.NotificationCompat$Extender {
+  public android.support.v4.app.NotificationCompat$CarExtender();
+  public android.support.v4.app.NotificationCompat$CarExtender(android.app.Notification);
+  public android.support.v4.app.NotificationCompat$Builder extend(android.support.v4.app.NotificationCompat$Builder);
+  public android.support.v4.app.NotificationCompat$CarExtender setColor(int);
+  public int getColor();
+  public android.support.v4.app.NotificationCompat$CarExtender setLargeIcon(android.graphics.Bitmap);
+  public android.graphics.Bitmap getLargeIcon();
+  public android.support.v4.app.NotificationCompat$CarExtender setUnreadConversation(android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation);
+  public android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation getUnreadConversation();
+}
diff -U 0 -N support-v4-21.0.0_55ef455a/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation support-v4-21.0.2_6496e6a2/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation
--- support-v4-21.0.0_55ef455a/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-21.0.2_6496e6a2/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,2 @@
+public class android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation extends android.support.v4.app.NotificationCompatBase$UnreadConversation {
+}
diff -U 0 -N support-v4-21.0.0_55ef455a/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder support-v4-21.0.2_6496e6a2/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder
--- support-v4-21.0.0_55ef455a/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-21.0.2_6496e6a2/android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder	2015-09-05 18:52:10.000000000 +0900
@@ -0,0 +1,8 @@
+public class android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder {
+  public android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder(java.lang.String);
+  public android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder addMessage(java.lang.String);
+  public android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder setReplyAction(android.app.PendingIntent, android.support.v4.app.RemoteInput);
+  public android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder setReadPendingIntent(android.app.PendingIntent);
+  public android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation$Builder setLatestTimestamp(long);
+  public android.support.v4.app.NotificationCompat$CarExtender$UnreadConversation build();
+}
diff -U 0 -N support-v4-21.0.0_55ef455a/android.support.v4.content.res.ResourcesCompat support-v4-21.0.2_6496e6a2/android.support.v4.content.res.ResourcesCompat
--- support-v4-21.0.0_55ef455a/android.support.v4.content.res.ResourcesCompat	2015-09-05 18:52:10.000000000 +0900
+++ support-v4-21.0.2_6496e6a2/android.support.v4.content.res.ResourcesCompat	2015-09-05 18:52:10.000000000 +0900
@@ -3 +3 @@
-  public android.graphics.drawable.Drawable getDrawable(android.content.res.Resources, int, android.content.res.Resources$Theme) throws android.content.res.Resources$NotFoundException;
+  public static android.graphics.drawable.Drawable getDrawable(android.content.res.Resources, int, android.content.res.Resources$Theme) throws android.content.res.Resources$NotFoundException;
