diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.ActivityCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.ActivityCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.ActivityCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.ActivityCompat	2016-05-16 18:11:08.000000000 +0900
@@ -5,0 +6 @@
+  public static void startIntentSenderForResult(android.app.Activity, android.content.IntentSender, int, android.content.Intent, int, int, int, android.os.Bundle) throws android.content.IntentSender$SendIntentException;
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.DialogFragment support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.DialogFragment
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.DialogFragment	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.DialogFragment	2016-05-16 18:11:08.000000000 +0900
@@ -18 +18 @@
-  public void onAttach(android.app.Activity);
+  public void onAttach(android.content.Context);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.Fragment support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.Fragment
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.Fragment	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.Fragment	2016-05-16 18:11:08.000000000 +0900
@@ -46,0 +47 @@
+  public void startIntentSenderForResult(android.content.IntentSender, int, android.content.Intent, int, int, int, android.os.Bundle) throws android.content.IntentSender$SendIntentException;
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.FragmentActivity support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.FragmentActivity
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.FragmentActivity	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.FragmentActivity	2016-05-16 18:11:08.000000000 +0900
@@ -31,0 +32 @@
+  public void startIntentSenderFromFragment(android.support.v4.app.Fragment, android.content.IntentSender, int, android.content.Intent, int, int, int, android.os.Bundle) throws android.content.IntentSender$SendIntentException;
@@ -33,0 +35,2 @@
+  public void startIntentSenderForResult(android.content.IntentSender, int, android.content.Intent, int, int, int, android.os.Bundle) throws android.content.IntentSender$SendIntentException;
+  public void startActivityForResult(android.content.Intent, int, android.os.Bundle);
@@ -34,0 +38 @@
+  public void startIntentSenderForResult(android.content.IntentSender, int, android.content.Intent, int, int, int) throws android.content.IntentSender$SendIntentException;
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.FragmentHostCallback<E> support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.FragmentHostCallback<E>
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.FragmentHostCallback<E>	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.FragmentHostCallback<E>	2016-05-16 18:11:08.000000000 +0900
@@ -9,0 +10 @@
+  public void onStartIntentSenderFromFragment(android.support.v4.app.Fragment, android.content.IntentSender, int, android.content.Intent, int, int, int, android.os.Bundle) throws android.content.IntentSender$SendIntentException;
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat	2016-05-16 18:11:08.000000000 +0900
@@ -25,0 +26 @@
+  public static final java.lang.String EXTRA_REMOTE_INPUT_HISTORY;
@@ -43,0 +45,3 @@
+  public static final java.lang.String EXTRA_SELF_DISPLAY_NAME;
+  public static final java.lang.String EXTRA_CONVERSATION_TITLE;
+  public static final java.lang.String EXTRA_MESSAGES;
@@ -59,0 +64 @@
+  public static final java.lang.String CATEGORY_REMINDER;
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Action support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Action
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Action	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Action	2016-05-16 18:11:08.000000000 +0900
@@ -10,0 +11 @@
+  public boolean getAllowGeneratedReplies();
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Action$Builder support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Action$Builder
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Action$Builder	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Action$Builder	2016-05-16 18:11:08.000000000 +0900
@@ -6,0 +7 @@
+  public android.support.v4.app.NotificationCompat$Action$Builder setAllowGeneratedReplies(boolean);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Action$WearableExtender support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Action$WearableExtender
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Action$WearableExtender	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Action$WearableExtender	2016-05-16 18:11:08.000000000 +0900
@@ -13,0 +14,2 @@
+  public android.support.v4.app.NotificationCompat$Action$WearableExtender setHintLaunchesActivity(boolean);
+  public boolean getHintLaunchesActivity();
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Builder support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Builder
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Builder	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Builder	2016-05-16 18:11:08.000000000 +0900
@@ -10,0 +11 @@
+  public java.lang.CharSequence[] mRemoteInputHistory;
@@ -22,0 +24 @@
+  public android.support.v4.app.NotificationCompat$Builder setRemoteInputHistory(java.lang.CharSequence[]);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$MessagingStyle support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$MessagingStyle
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$MessagingStyle	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$MessagingStyle	2016-05-16 18:11:08.000000000 +0900
@@ -0,0 +1,12 @@
+public class android.support.v4.app.NotificationCompat$MessagingStyle extends android.support.v4.app.NotificationCompat$Style {
+  public static final int MAXIMUM_RETAINED_MESSAGES;
+  public android.support.v4.app.NotificationCompat$MessagingStyle(java.lang.CharSequence);
+  public java.lang.CharSequence getUserDisplayName();
+  public android.support.v4.app.NotificationCompat$MessagingStyle setConversationTitle(java.lang.CharSequence);
+  public java.lang.CharSequence getConversationTitle();
+  public android.support.v4.app.NotificationCompat$MessagingStyle addMessage(java.lang.CharSequence, long, java.lang.CharSequence);
+  public android.support.v4.app.NotificationCompat$MessagingStyle addMessage(android.support.v4.app.NotificationCompat$MessagingStyle$Message);
+  public java.util.List<android.support.v4.app.NotificationCompat$MessagingStyle$Message> getMessages();
+  public static android.support.v4.app.NotificationCompat$MessagingStyle extractMessagingStyleFromNotification(android.app.Notification);
+  public void addCompatExtras(android.os.Bundle);
+}
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$MessagingStyle$Message support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$MessagingStyle$Message
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$MessagingStyle$Message	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$MessagingStyle$Message	2016-05-16 18:11:08.000000000 +0900
@@ -0,0 +1,9 @@
+public final class android.support.v4.app.NotificationCompat$MessagingStyle$Message {
+  public android.support.v4.app.NotificationCompat$MessagingStyle$Message(java.lang.CharSequence, long, java.lang.CharSequence);
+  public android.support.v4.app.NotificationCompat$MessagingStyle$Message setData(java.lang.String, android.net.Uri);
+  public java.lang.CharSequence getText();
+  public long getTimestamp();
+  public java.lang.CharSequence getSender();
+  public java.lang.String getDataMimeType();
+  public android.net.Uri getDataUri();
+}
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Style support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Style
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$Style	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$Style	2016-05-16 18:11:08.000000000 +0900
@@ -4,0 +5 @@
+  public void addCompatExtras(android.os.Bundle);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$WearableExtender support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$WearableExtender
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationCompat$WearableExtender	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationCompat$WearableExtender	2016-05-16 18:11:08.000000000 +0900
@@ -50,0 +51,6 @@
+  public android.support.v4.app.NotificationCompat$WearableExtender setHintAmbientBigPicture(boolean);
+  public boolean getHintAmbientBigPicture();
+  public android.support.v4.app.NotificationCompat$WearableExtender setHintContentIntentLaunchesActivity(boolean);
+  public boolean getHintContentIntentLaunchesActivity();
+  public android.support.v4.app.NotificationCompat$WearableExtender setDismissalId(java.lang.String);
+  public java.lang.String getDismissalId();
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationManagerCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationManagerCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.app.NotificationManagerCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.app.NotificationManagerCompat	2016-05-16 18:11:08.000000000 +0900
@@ -9,0 +10 @@
+  public boolean areNotificationsEnabled();
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.content.ContextCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.content.ContextCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.content.ContextCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.content.ContextCompat	2016-05-16 18:11:08.000000000 +0900
@@ -13 +13 @@
-  public final java.io.File getNoBackupFilesDir(android.content.Context);
+  public static final java.io.File getNoBackupFilesDir(android.content.Context);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.graphics.drawable.DrawableCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.graphics.drawable.DrawableCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.graphics.drawable.DrawableCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.graphics.drawable.DrawableCompat	2016-05-16 18:11:08.000000000 +0900
@@ -17 +17 @@
-  public static void setLayoutDirection(android.graphics.drawable.Drawable, int);
+  public static boolean setLayoutDirection(android.graphics.drawable.Drawable, int);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.media.MediaBrowserCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.media.MediaBrowserCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.media.MediaBrowserCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.media.MediaBrowserCompat	2016-05-16 18:11:08.000000000 +0900
@@ -15 +15 @@
-  public void unsubscribe(java.lang.String, android.os.Bundle);
+  public void unsubscribe(java.lang.String, android.support.v4.media.MediaBrowserCompat$SubscriptionCallback);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.media.MediaBrowserCompatUtils support-v4-24.0.0-beta1_5217cb4e/android.support.v4.media.MediaBrowserCompatUtils
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.media.MediaBrowserCompatUtils	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.media.MediaBrowserCompatUtils	2016-05-16 18:11:08.000000000 +0900
@@ -5 +4,0 @@
-  public static java.util.List<android.support.v4.media.MediaBrowserCompat$MediaItem> applyOptions(java.util.List<android.support.v4.media.MediaBrowserCompat$MediaItem>, android.os.Bundle);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.media.MediaBrowserServiceCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.media.MediaBrowserServiceCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.media.MediaBrowserServiceCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.media.MediaBrowserServiceCompat	2016-05-16 18:11:08.000000000 +0900
@@ -13,0 +14 @@
+  public final android.os.Bundle getBrowserRootHints();
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.AbsSavedState support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.AbsSavedState
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.AbsSavedState	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.AbsSavedState	2016-05-16 18:11:08.000000000 +0900
@@ -0,0 +1,7 @@
+public abstract class android.support.v4.view.AbsSavedState implements android.os.Parcelable {
+  public static final android.support.v4.view.AbsSavedState EMPTY_STATE;
+  public static final android.os.Parcelable$Creator<android.support.v4.view.AbsSavedState> CREATOR;
+  public final android.os.Parcelable getSuperState();
+  public int describeContents();
+  public void writeToParcel(android.os.Parcel, int);
+}
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.PagerTitleStrip support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.PagerTitleStrip
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.PagerTitleStrip	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.PagerTitleStrip	2016-05-16 18:11:08.000000000 +0900
@@ -1 +1 @@
-public class android.support.v4.view.PagerTitleStrip extends android.view.ViewGroup implements android.support.v4.view.ViewPager$Decor {
+public class android.support.v4.view.PagerTitleStrip extends android.view.ViewGroup {
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewCompat support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewCompat
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewCompat	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewCompat	2016-05-16 18:11:08.000000000 +0900
@@ -74,0 +75 @@
+  public static android.graphics.Matrix getMatrix(android.view.View);
@@ -130,0 +132 @@
+  public static boolean isInLayout(android.view.View);
@@ -131,0 +134 @@
+  public static boolean isLayoutDirectionResolved(android.view.View);
@@ -132,0 +136 @@
+  public static void setZ(android.view.View, float);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager	2016-05-16 18:11:08.000000000 +0900
@@ -8,0 +9,2 @@
+  public void addOnAdapterChangeListener(android.support.v4.view.ViewPager$OnAdapterChangeListener);
+  public void removeOnAdapterChangeListener(android.support.v4.view.ViewPager$OnAdapterChangeListener);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager$DecorView support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager$DecorView
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager$DecorView	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager$DecorView	2016-05-16 18:11:08.000000000 +0900
@@ -0,0 +1,2 @@
+public interface android.support.v4.view.ViewPager$DecorView extends java.lang.annotation.Annotation {
+}
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager$OnAdapterChangeListener support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager$OnAdapterChangeListener
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager$OnAdapterChangeListener	1970-01-01 09:00:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager$OnAdapterChangeListener	2016-05-16 18:11:08.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v4.view.ViewPager$OnAdapterChangeListener {
+  public abstract void onAdapterChanged(android.support.v4.view.ViewPager, android.support.v4.view.PagerAdapter, android.support.v4.view.PagerAdapter);
+}
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager$SavedState support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager$SavedState
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.view.ViewPager$SavedState	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.view.ViewPager$SavedState	2016-05-16 18:11:08.000000000 +0900
@@ -1 +1 @@
-public class android.support.v4.view.ViewPager$SavedState extends android.view.View$BaseSavedState {
+public class android.support.v4.view.ViewPager$SavedState extends android.support.v4.view.AbsSavedState {
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.widget.DrawerLayout support-v4-24.0.0-beta1_5217cb4e/android.support.v4.widget.DrawerLayout
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.widget.DrawerLayout	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.widget.DrawerLayout	2016-05-16 18:11:08.000000000 +0900
@@ -40,0 +41 @@
+  public void openDrawer(android.view.View, boolean);
@@ -41,0 +43 @@
+  public void openDrawer(int, boolean);
@@ -42,0 +45 @@
+  public void closeDrawer(android.view.View, boolean);
@@ -43,0 +47 @@
+  public void closeDrawer(int, boolean);
diff -U 0 -N support-v4-24.0.0-alpha2_d29bf872/android.support.v4.widget.DrawerLayout$SavedState support-v4-24.0.0-beta1_5217cb4e/android.support.v4.widget.DrawerLayout$SavedState
--- support-v4-24.0.0-alpha2_d29bf872/android.support.v4.widget.DrawerLayout$SavedState	2016-05-16 18:10:00.000000000 +0900
+++ support-v4-24.0.0-beta1_5217cb4e/android.support.v4.widget.DrawerLayout$SavedState	2016-05-16 18:11:08.000000000 +0900
@@ -1 +1 @@
-public class android.support.v4.widget.DrawerLayout$SavedState extends android.view.View$BaseSavedState {
+public class android.support.v4.widget.DrawerLayout$SavedState extends android.support.v4.view.AbsSavedState {
@@ -3 +3 @@
-  public android.support.v4.widget.DrawerLayout$SavedState(android.os.Parcel);
+  public android.support.v4.widget.DrawerLayout$SavedState(android.os.Parcel, java.lang.ClassLoader);
