diff -U 0 -N support-vector-drawable-24.2.1_fbda778e/android.support.graphics.drawable.VectorDrawableCompat support-vector-drawable-25.0.0_e5111f3e/android.support.graphics.drawable.VectorDrawableCompat
--- support-vector-drawable-24.2.1_fbda778e/android.support.graphics.drawable.VectorDrawableCompat	2016-10-21 01:01:15.000000000 +0900
+++ support-vector-drawable-25.0.0_e5111f3e/android.support.graphics.drawable.VectorDrawableCompat	2016-10-21 01:01:15.000000000 +0900
@@ -15,0 +16,2 @@
+  public boolean isAutoMirrored();
+  public void setAutoMirrored(boolean);
@@ -35 +36,0 @@
-  public int getLayoutDirection();
@@ -37,2 +37,0 @@
-  public boolean isAutoMirrored();
-  public void setAutoMirrored(boolean);
