diff -U 0 -N palette-v7-22.0.0_c3c3a5ee/android.support.v7.graphics.Palette palette-v7-22.1.0_e346e04a/android.support.v7.graphics.Palette
--- palette-v7-22.0.0_c3c3a5ee/android.support.v7.graphics.Palette	2015-08-20 23:41:45.000000000 +0900
+++ palette-v7-22.1.0_e346e04a/android.support.v7.graphics.Palette	2015-08-20 23:41:45.000000000 +0900
@@ -1,0 +2,2 @@
+  public static android.support.v7.graphics.Palette$Builder from(android.graphics.Bitmap);
+  public static android.support.v7.graphics.Palette from(java.util.List<android.support.v7.graphics.Palette$Swatch>);
@@ -6 +7,0 @@
-  public static android.support.v7.graphics.Palette from(java.util.List<android.support.v7.graphics.Palette$Swatch>);
@@ -20,2 +20,0 @@
-  public boolean equals(java.lang.Object);
-  public int hashCode();
diff -U 0 -N palette-v7-22.0.0_c3c3a5ee/android.support.v7.graphics.Palette$Builder palette-v7-22.1.0_e346e04a/android.support.v7.graphics.Palette$Builder
--- palette-v7-22.0.0_c3c3a5ee/android.support.v7.graphics.Palette$Builder	1970-01-01 09:00:00.000000000 +0900
+++ palette-v7-22.1.0_e346e04a/android.support.v7.graphics.Palette$Builder	2015-08-20 23:41:45.000000000 +0900
@@ -0,0 +1,8 @@
+public final class android.support.v7.graphics.Palette$Builder {
+  public android.support.v7.graphics.Palette$Builder(android.graphics.Bitmap);
+  public android.support.v7.graphics.Palette$Builder(java.util.List<android.support.v7.graphics.Palette$Swatch>);
+  public android.support.v7.graphics.Palette$Builder maximumColorCount(int);
+  public android.support.v7.graphics.Palette$Builder resizeBitmapSize(int);
+  public android.support.v7.graphics.Palette generate();
+  public android.os.AsyncTask<android.graphics.Bitmap, java.lang.Void, android.support.v7.graphics.Palette> generate(android.support.v7.graphics.Palette$PaletteAsyncListener);
+}
