diff -U 0 -N mediarouter-v7-21.0.3_6b4151da/android.support.v7.app.MediaRouteControllerDialog mediarouter-v7-22.0.0_7fe297a6/android.support.v7.app.MediaRouteControllerDialog
--- mediarouter-v7-21.0.3_6b4151da/android.support.v7.app.MediaRouteControllerDialog	2015-08-20 23:41:45.000000000 +0900
+++ mediarouter-v7-22.0.0_7fe297a6/android.support.v7.app.MediaRouteControllerDialog	2015-08-20 23:41:45.000000000 +0900
@@ -7,2 +7 @@
-  public void setVolumeControlEnabled(boolean);
-  public boolean isVolumeControlEnabled();
+  public android.support.v4.media.session.MediaSessionCompat$Token getMediaSession();
diff -U 0 -N mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouteDescriptor mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouteDescriptor
--- mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouteDescriptor	2015-08-20 23:41:45.000000000 +0900
+++ mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouteDescriptor	2015-08-20 23:41:45.000000000 +0900
@@ -6,0 +7,2 @@
+  public boolean canDisconnectAndKeepPlaying();
+  public android.content.IntentSender getSettingsActivity();
diff -U 0 -N mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouteDescriptor$Builder mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouteDescriptor$Builder
--- mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouteDescriptor$Builder	2015-08-20 23:41:45.000000000 +0900
+++ mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouteDescriptor$Builder	2015-08-20 23:41:45.000000000 +0900
@@ -8,0 +9,2 @@
+  public android.support.v7.media.MediaRouteDescriptor$Builder setCanDisconnect(boolean);
+  public android.support.v7.media.MediaRouteDescriptor$Builder setSettingsActivity(android.content.IntentSender);
diff -U 0 -N mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouteProvider$RouteController mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouteProvider$RouteController
--- mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouteProvider$RouteController	2015-08-20 23:41:45.000000000 +0900
+++ mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouteProvider$RouteController	2015-08-20 23:41:45.000000000 +0900
@@ -5,0 +6 @@
+  public void onUnselect(int);
diff -U 0 -N mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouter mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouter
--- mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouter	2015-08-20 23:41:45.000000000 +0900
+++ mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouter	2015-08-20 23:41:45.000000000 +0900
@@ -1,0 +2,4 @@
+  public static final int UNSELECT_REASON_UNKNOWN;
+  public static final int UNSELECT_REASON_DISCONNECTED;
+  public static final int UNSELECT_REASON_STOPPED;
+  public static final int UNSELECT_REASON_ROUTE_CHANGED;
@@ -14,0 +19 @@
+  public void unselect(int);
@@ -23,0 +29,2 @@
+  public void setMediaSessionCompat(android.support.v4.media.session.MediaSessionCompat);
+  public android.support.v4.media.session.MediaSessionCompat$Token getMediaSessionToken();
diff -U 0 -N mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouter$RouteInfo mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouter$RouteInfo
--- mediarouter-v7-21.0.3_6b4151da/android.support.v7.media.MediaRouter$RouteInfo	2015-08-20 23:41:45.000000000 +0900
+++ mediarouter-v7-22.0.0_7fe297a6/android.support.v7.media.MediaRouter$RouteInfo	2015-08-20 23:41:45.000000000 +0900
@@ -24,0 +25 @@
+  public boolean canDisconnect();
@@ -28,0 +30 @@
+  public android.content.IntentSender getSettingsIntent();
diff -U 0 -N mediarouter-v7-21.0.3_6b4151da/android.support.v7.mediarouter.BuildConfig mediarouter-v7-22.0.0_7fe297a6/android.support.v7.mediarouter.BuildConfig
--- mediarouter-v7-21.0.3_6b4151da/android.support.v7.mediarouter.BuildConfig	2015-08-20 23:41:45.000000000 +0900
+++ mediarouter-v7-22.0.0_7fe297a6/android.support.v7.mediarouter.BuildConfig	2015-08-20 23:41:45.000000000 +0900
@@ -3 +3 @@
-  public static final java.lang.String PACKAGE_NAME;
+  public static final java.lang.String APPLICATION_ID;
