diff -U 0 -N mediarouter-v7-23.0.1_d33975ba/android.support.v7.app.MediaRouteChooserDialogFragment mediarouter-v7-23.1.0_100fac2a/android.support.v7.app.MediaRouteChooserDialogFragment
--- mediarouter-v7-23.0.1_d33975ba/android.support.v7.app.MediaRouteChooserDialogFragment	2015-10-17 20:26:40.000000000 +0900
+++ mediarouter-v7-23.1.0_100fac2a/android.support.v7.app.MediaRouteChooserDialogFragment	2015-10-17 20:26:38.000000000 +0900
@@ -6,0 +7 @@
+  public void onConfigurationChanged(android.content.res.Configuration);
diff -U 0 -N mediarouter-v7-23.0.1_d33975ba/android.support.v7.app.MediaRouteControllerDialogFragment mediarouter-v7-23.1.0_100fac2a/android.support.v7.app.MediaRouteControllerDialogFragment
--- mediarouter-v7-23.0.1_d33975ba/android.support.v7.app.MediaRouteControllerDialogFragment	2015-10-17 20:26:40.000000000 +0900
+++ mediarouter-v7-23.1.0_100fac2a/android.support.v7.app.MediaRouteControllerDialogFragment	2015-10-17 20:26:38.000000000 +0900
@@ -4,0 +5 @@
+  public void onConfigurationChanged(android.content.res.Configuration);
diff -U 0 -N mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouteDescriptor mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouteDescriptor
--- mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouteDescriptor	2015-10-17 20:26:40.000000000 +0900
+++ mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouteDescriptor	2015-10-17 20:26:38.000000000 +0900
@@ -2,0 +3 @@
+  public java.util.List<java.lang.String> getGroupMemberIds();
@@ -4,0 +6 @@
+  public android.net.Uri getIconUri();
@@ -6,0 +9 @@
+  public int getConnectionState();
@@ -11,0 +15 @@
+  public int getDeviceType();
diff -U 0 -N mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouteDescriptor$Builder mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouteDescriptor$Builder
--- mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouteDescriptor$Builder	2015-10-17 20:26:40.000000000 +0900
+++ mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouteDescriptor$Builder	2015-10-17 20:26:38.000000000 +0900
@@ -4,0 +5,2 @@
+  public android.support.v7.media.MediaRouteDescriptor$Builder addGroupMemberId(java.lang.String);
+  public android.support.v7.media.MediaRouteDescriptor$Builder addGroupMemberIds(java.util.Collection<java.lang.String>);
@@ -6,0 +9 @@
+  public android.support.v7.media.MediaRouteDescriptor$Builder setIconUri(android.net.Uri);
@@ -8,0 +12 @@
+  public android.support.v7.media.MediaRouteDescriptor$Builder setConnectionState(int);
@@ -14,0 +19 @@
+  public android.support.v7.media.MediaRouteDescriptor$Builder setDeviceType(int);
diff -U 0 -N mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouter$RouteGroup mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouter$RouteGroup
--- mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouter$RouteGroup	1970-01-01 09:00:00.000000000 +0900
+++ mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouter$RouteGroup	2015-10-17 20:26:38.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v7.media.MediaRouter$RouteGroup extends android.support.v7.media.MediaRouter$RouteInfo {
+  public int getRouteCount();
+  public android.support.v7.media.MediaRouter$RouteInfo getRouteAt(int);
+  public java.util.List<android.support.v7.media.MediaRouter$RouteInfo> getRoutes();
+  public java.lang.String toString();
+}
diff -U 0 -N mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouter$RouteInfo mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouter$RouteInfo
--- mediarouter-v7-23.0.1_d33975ba/android.support.v7.media.MediaRouter$RouteInfo	2015-10-17 20:26:40.000000000 +0900
+++ mediarouter-v7-23.1.0_100fac2a/android.support.v7.media.MediaRouter$RouteInfo	2015-10-17 20:26:38.000000000 +0900
@@ -1 +1,4 @@
-public final class android.support.v7.media.MediaRouter$RouteInfo {
+public class android.support.v7.media.MediaRouter$RouteInfo {
+  public static final int CONNECTION_STATE_DISCONNECTED;
+  public static final int CONNECTION_STATE_CONNECTING;
+  public static final int CONNECTION_STATE_CONNECTED;
@@ -3,0 +7,4 @@
+  public static final int DEVICE_TYPE_UNKNOWN;
+  public static final int DEVICE_TYPE_BLUETOOTH;
+  public static final int DEVICE_TYPE_TV;
+  public static final int DEVICE_TYPE_SPEAKER;
@@ -5,0 +13 @@
+  public static final int PRESENTATION_DISPLAY_ID_NONE;
@@ -9,0 +18 @@
+  public android.net.Uri getIconUri();
@@ -11,0 +21 @@
+  public int getConnectionState();
@@ -21,0 +32 @@
+  public int getDeviceType();
@@ -28,0 +40 @@
+  public int getPresentationDisplayId();
@@ -32,0 +45 @@
+  public android.support.v7.media.MediaRouteProvider getProviderInstance();
