diff -U 0 -N mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.MediaControlIntent mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.MediaControlIntent
--- mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.MediaControlIntent	2016-04-16 19:37:19.000000000 +0900
+++ mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.MediaControlIntent	2016-04-16 19:37:19.000000000 +0900
@@ -16 +16 @@
-  public static final java.lang.String ACTION_SEND_CUSTOM_MESSAGE;
+  public static final java.lang.String ACTION_SEND_MESSAGE;
@@ -20 +20 @@
-  public static final java.lang.String EXTRA_CUSTOM_MESSAGE_RECEIVER;
+  public static final java.lang.String EXTRA_MESSAGE_RECEIVER;
@@ -27 +27 @@
-  public static final java.lang.String EXTRA_CUSTOM_MESSAGE;
+  public static final java.lang.String EXTRA_MESSAGE;
@@ -33 +32,0 @@
-  public static final java.lang.String buildCustomCategory(java.lang.String);
diff -U 0 -N mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.MediaRouteProviderService mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.MediaRouteProviderService
--- mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.MediaRouteProviderService	2016-04-16 19:37:19.000000000 +0900
+++ mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.MediaRouteProviderService	2016-04-16 19:37:19.000000000 +0900
@@ -6,0 +7 @@
+  public boolean onUnbind(android.content.Intent);
diff -U 0 -N mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.RemotePlaybackClient mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.RemotePlaybackClient
--- mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.RemotePlaybackClient	2016-04-16 19:37:19.000000000 +0900
+++ mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.RemotePlaybackClient	2016-04-16 19:37:19.000000000 +0900
@@ -7 +7 @@
-  public boolean isCustomMessageSupported();
+  public boolean isMessagingSupported();
@@ -12 +12 @@
-  public void setCustomMessageCallback(android.support.v7.media.RemotePlaybackClient$CustomMessageCallback);
+  public void setMessageCallback(android.support.v7.media.RemotePlaybackClient$MessageCallback);
@@ -22 +22 @@
-  public void sendCustomMessage(android.os.Bundle, android.support.v7.media.RemotePlaybackClient$SessionActionCallback);
+  public void sendMessage(android.os.Bundle, android.support.v7.media.RemotePlaybackClient$SessionActionCallback);
diff -U 0 -N mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.RemotePlaybackClient$CustomMessageCallback mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.RemotePlaybackClient$CustomMessageCallback
--- mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.RemotePlaybackClient$CustomMessageCallback	2016-04-16 19:37:19.000000000 +0900
+++ mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.RemotePlaybackClient$CustomMessageCallback	1970-01-01 09:00:00.000000000 +0900
@@ -1,4 +0,0 @@
-public abstract class android.support.v7.media.RemotePlaybackClient$CustomMessageCallback {
-  public android.support.v7.media.RemotePlaybackClient$CustomMessageCallback();
-  public void onMessageReceived(java.lang.String, android.os.Bundle);
-}
diff -U 0 -N mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.RemotePlaybackClient$MessageCallback mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.RemotePlaybackClient$MessageCallback
--- mediarouter-v7-24.0.0-alpha1_94d3d956/android.support.v7.media.RemotePlaybackClient$MessageCallback	1970-01-01 09:00:00.000000000 +0900
+++ mediarouter-v7-24.0.0-alpha2_850a6810/android.support.v7.media.RemotePlaybackClient$MessageCallback	2016-04-16 19:37:19.000000000 +0900
@@ -0,0 +1,4 @@
+public abstract class android.support.v7.media.RemotePlaybackClient$MessageCallback {
+  public android.support.v7.media.RemotePlaybackClient$MessageCallback();
+  public void onMessageReceived(java.lang.String, android.os.Bundle);
+}
