diff -U 0 -N cardview-v7-23.3.0_62811ea2/android.support.v7.widget.CardView cardview-v7-23.4.0_393d23b8/android.support.v7.widget.CardView
--- cardview-v7-23.3.0_62811ea2/android.support.v7.widget.CardView	2016-05-13 21:21:42.000000000 +0900
+++ cardview-v7-23.4.0_393d23b8/android.support.v7.widget.CardView	2016-05-13 21:21:42.000000000 +0900
@@ -1 +1 @@
-public class android.support.v7.widget.CardView extends android.widget.FrameLayout implements android.support.v7.widget.CardViewDelegate {
+public class android.support.v7.widget.CardView extends android.widget.FrameLayout {
@@ -12 +11,0 @@
-  public void setMinWidthHeightInternal(int, int);
@@ -20 +18,0 @@
-  public void setShadowPadding(int, int, int, int);
