diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.BuildConfig leanback-v17-22.0.0_5260278a/android.support.v17.leanback.BuildConfig
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.BuildConfig	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.BuildConfig	2015-08-20 23:41:43.000000000 +0900
@@ -3 +3 @@
-  public static final java.lang.String PACKAGE_NAME;
+  public static final java.lang.String APPLICATION_ID;
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.BrowseFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.BrowseFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.BrowseFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.BrowseFragment	2015-08-20 23:41:43.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.BrowseFragment extends android.app.Fragment {
+public class android.support.v17.leanback.app.BrowseFragment extends android.support.v17.leanback.app.BaseFragment {
@@ -32,0 +33,2 @@
+  public void setSelectedPosition(int);
+  public void setSelectedPosition(int, boolean);
@@ -33,0 +36,2 @@
+  public void onPause();
+  public void onResume();
@@ -41,0 +46,3 @@
+  public void startEntranceTransition();
+  public void prepareEntranceTransition();
+  public void onViewCreated(android.view.View, android.os.Bundle);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.BrowseSupportFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.BrowseSupportFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.BrowseSupportFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.BrowseSupportFragment	2015-08-20 23:41:43.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.BrowseSupportFragment extends android.support.v4.app.Fragment {
+public class android.support.v17.leanback.app.BrowseSupportFragment extends android.support.v17.leanback.app.BaseSupportFragment {
@@ -32,0 +33,2 @@
+  public void setSelectedPosition(int);
+  public void setSelectedPosition(int, boolean);
@@ -33,0 +36,2 @@
+  public void onPause();
+  public void onResume();
@@ -41,0 +46,3 @@
+  public void startEntranceTransition();
+  public void prepareEntranceTransition();
+  public void onViewCreated(android.view.View, android.os.Bundle);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.DetailsFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.DetailsFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.DetailsFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.DetailsFragment	2015-08-20 23:41:43.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.DetailsFragment extends android.app.Fragment {
+public class android.support.v17.leanback.app.DetailsFragment extends android.support.v17.leanback.app.BaseFragment {
@@ -12,0 +13,2 @@
+  public void setSelectedPosition(int);
+  public void setSelectedPosition(int, boolean);
@@ -13,0 +16,3 @@
+  public void startEntranceTransition();
+  public void prepareEntranceTransition();
+  public void onViewCreated(android.view.View, android.os.Bundle);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.DetailsSupportFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.DetailsSupportFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.DetailsSupportFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.DetailsSupportFragment	2015-08-20 23:41:43.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.DetailsSupportFragment extends android.support.v4.app.Fragment {
+public class android.support.v17.leanback.app.DetailsSupportFragment extends android.support.v17.leanback.app.BaseSupportFragment {
@@ -12,0 +13,2 @@
+  public void setSelectedPosition(int);
+  public void setSelectedPosition(int, boolean);
@@ -13,0 +16,3 @@
+  public void startEntranceTransition();
+  public void prepareEntranceTransition();
+  public void onViewCreated(android.view.View, android.os.Bundle);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.HeadersFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.HeadersFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.HeadersFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.HeadersFragment	2015-08-20 23:41:43.000000000 +0900
@@ -5,0 +6 @@
+  public void setSelectedPosition(int, boolean);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.HeadersSupportFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.HeadersSupportFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.HeadersSupportFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.HeadersSupportFragment	2015-08-20 23:41:43.000000000 +0900
@@ -5,0 +6 @@
+  public void setSelectedPosition(int, boolean);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.MediaControllerGlue leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.MediaControllerGlue
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.MediaControllerGlue	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.MediaControllerGlue	2015-08-20 23:41:43.000000000 +0900
@@ -0,0 +1,16 @@
+public abstract class android.support.v17.leanback.app.MediaControllerGlue extends android.support.v17.leanback.app.PlaybackControlGlue {
+  public android.support.v17.leanback.app.MediaControllerGlue(android.content.Context, android.support.v17.leanback.app.PlaybackOverlayFragment, int[]);
+  public android.support.v17.leanback.app.MediaControllerGlue(android.content.Context, android.support.v17.leanback.app.PlaybackOverlayFragment, int[], int[]);
+  public void attachToMediaController(android.support.v4.media.session.MediaControllerCompat);
+  public void detach();
+  public final android.support.v4.media.session.MediaControllerCompat getMediaController();
+  public boolean hasValidMedia();
+  public boolean isMediaPlaying();
+  public int getCurrentSpeedId();
+  public java.lang.CharSequence getMediaTitle();
+  public java.lang.CharSequence getMediaSubtitle();
+  public int getMediaDuration();
+  public int getCurrentPosition();
+  public android.graphics.drawable.Drawable getMediaArt();
+  public long getSupportedActions();
+}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackControlGlue leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackControlGlue
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackControlGlue	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackControlGlue	2015-08-20 23:41:43.000000000 +0900
@@ -0,0 +1,42 @@
+public abstract class android.support.v17.leanback.app.PlaybackControlGlue {
+  public static final int ACTION_CUSTOM_LEFT_FIRST;
+  public static final int ACTION_SKIP_TO_PREVIOUS;
+  public static final int ACTION_REWIND;
+  public static final int ACTION_PLAY_PAUSE;
+  public static final int ACTION_FAST_FORWARD;
+  public static final int ACTION_SKIP_TO_NEXT;
+  public static final int ACTION_CUSTOM_RIGHT_FIRST;
+  public static final int PLAYBACK_SPEED_INVALID;
+  public static final int PLAYBACK_SPEED_PAUSED;
+  public static final int PLAYBACK_SPEED_NORMAL;
+  public static final int PLAYBACK_SPEED_FAST_L0;
+  public static final int PLAYBACK_SPEED_FAST_L1;
+  public static final int PLAYBACK_SPEED_FAST_L2;
+  public static final int PLAYBACK_SPEED_FAST_L3;
+  public static final int PLAYBACK_SPEED_FAST_L4;
+  public android.support.v17.leanback.app.PlaybackControlGlue(android.content.Context, android.support.v17.leanback.app.PlaybackOverlayFragment, int[]);
+  public android.support.v17.leanback.app.PlaybackControlGlue(android.content.Context, android.support.v17.leanback.app.PlaybackOverlayFragment, int[], int[]);
+  public android.support.v17.leanback.widget.PlaybackControlsRowPresenter createControlsRowAndPresenter();
+  public android.support.v17.leanback.app.PlaybackOverlayFragment getFragment();
+  public android.content.Context getContext();
+  public int[] getFastForwardSpeeds();
+  public int[] getRewindSpeeds();
+  public void setFadingEnabled(boolean);
+  public boolean isFadingEnabled();
+  public void setOnItemViewClickedListener(android.support.v17.leanback.widget.OnItemViewClickedListener);
+  public android.support.v17.leanback.widget.OnItemViewClickedListener getOnItemViewClickedListener();
+  public void setControlsRow(android.support.v17.leanback.widget.PlaybackControlsRow);
+  public android.support.v17.leanback.widget.PlaybackControlsRow getControlsRow();
+  public void enableProgressUpdating(boolean);
+  public int getUpdatePeriod();
+  public void updateProgress();
+  public abstract boolean hasValidMedia();
+  public abstract boolean isMediaPlaying();
+  public abstract java.lang.CharSequence getMediaTitle();
+  public abstract java.lang.CharSequence getMediaSubtitle();
+  public abstract int getMediaDuration();
+  public abstract android.graphics.drawable.Drawable getMediaArt();
+  public abstract long getSupportedActions();
+  public abstract int getCurrentSpeedId();
+  public abstract int getCurrentPosition();
+}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlayFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlayFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlayFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlayFragment	2015-08-20 23:41:43.000000000 +0900
@@ -9,0 +10,2 @@
+  public final void setInputEventHandler(android.support.v17.leanback.app.PlaybackOverlayFragment$InputEventHandler);
+  public final android.support.v17.leanback.app.PlaybackOverlayFragment$InputEventHandler getInputEventHandler();
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlayFragment$InputEventHandler leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlayFragment$InputEventHandler
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlayFragment$InputEventHandler	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlayFragment$InputEventHandler	2015-08-20 23:41:43.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v17.leanback.app.PlaybackOverlayFragment$InputEventHandler {
+  public abstract boolean handleInputEvent(android.view.InputEvent);
+}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment	2015-08-20 23:41:43.000000000 +0900
@@ -9,0 +10,2 @@
+  public final void setInputEventHandler(android.support.v17.leanback.app.PlaybackOverlaySupportFragment$InputEventHandler);
+  public final android.support.v17.leanback.app.PlaybackOverlaySupportFragment$InputEventHandler getInputEventHandler();
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment$InputEventHandler leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment$InputEventHandler
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment$InputEventHandler	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.PlaybackOverlaySupportFragment$InputEventHandler	2015-08-20 23:41:43.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v17.leanback.app.PlaybackOverlaySupportFragment$InputEventHandler {
+  public abstract boolean handleInputEvent(android.view.InputEvent);
+}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.RowsFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.RowsFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.RowsFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.RowsFragment	2015-08-20 23:41:43.000000000 +0900
@@ -15,0 +16 @@
+  public void setSelectedPosition(int, boolean);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.RowsSupportFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.RowsSupportFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.RowsSupportFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.RowsSupportFragment	2015-08-20 23:41:43.000000000 +0900
@@ -15,0 +16 @@
+  public void setSelectedPosition(int, boolean);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.VerticalGridFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.VerticalGridFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.VerticalGridFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.VerticalGridFragment	2015-08-20 23:41:43.000000000 +0900
@@ -24,0 +25,2 @@
+  public void onPause();
+  public void onResume();
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.VerticalGridSupportFragment leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.VerticalGridSupportFragment
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.app.VerticalGridSupportFragment	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.app.VerticalGridSupportFragment	2015-08-20 23:41:43.000000000 +0900
@@ -24,0 +25,2 @@
+  public void onPause();
+  public void onResume();
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.LeanbackTransitionHelper leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.LeanbackTransitionHelper
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.LeanbackTransitionHelper	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.LeanbackTransitionHelper	2015-08-20 23:41:43.000000000 +0900
@@ -0,0 +1,5 @@
+public class android.support.v17.leanback.transition.LeanbackTransitionHelper {
+  public android.support.v17.leanback.transition.LeanbackTransitionHelper();
+  public static java.lang.Object loadTitleInTransition(android.content.Context, android.support.v17.leanback.transition.TransitionHelper);
+  public static java.lang.Object loadTitleOutTransition(android.content.Context, android.support.v17.leanback.transition.TransitionHelper);
+}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.SlideCallback leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.SlideCallback
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.SlideCallback	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.SlideCallback	1970-01-01 09:00:00.000000000 +0900
@@ -1,3 +0,0 @@
-public interface android.support.v17.leanback.transition.SlideCallback {
-  public abstract boolean getSlide(android.view.View, boolean, int[], float[]);
-}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.SlideNoPropagation leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.SlideNoPropagation
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.SlideNoPropagation	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.SlideNoPropagation	2015-08-20 23:41:43.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v17.leanback.transition.SlideNoPropagation extends android.transition.Slide {
+  public android.support.v17.leanback.transition.SlideNoPropagation();
+  public android.support.v17.leanback.transition.SlideNoPropagation(int);
+  public android.support.v17.leanback.transition.SlideNoPropagation(android.content.Context, android.util.AttributeSet);
+  public void setSlideEdge(int);
+}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.TransitionHelper leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.TransitionHelper
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.transition.TransitionHelper	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.transition.TransitionHelper	2015-08-20 23:41:43.000000000 +0900
@@ -8,0 +9 @@
+  public static boolean systemSupportsEntranceTransitions();
@@ -25 +26 @@
-  public java.lang.Object createSlide(android.support.v17.leanback.transition.SlideCallback);
+  public java.lang.Object createSlide(int);
@@ -42,0 +44 @@
+  public java.lang.Object loadTransition(android.content.Context, int);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.ArrayObjectAdapter leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.ArrayObjectAdapter
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.ArrayObjectAdapter	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.ArrayObjectAdapter	2015-08-20 23:41:43.000000000 +0900
@@ -12,0 +13 @@
+  public void replace(int, java.lang.Object);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.HeaderItem leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.HeaderItem
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.HeaderItem	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.HeaderItem	2015-08-20 23:41:43.000000000 +0900
@@ -2,2 +2,2 @@
-  public android.support.v17.leanback.widget.HeaderItem(long, java.lang.String, java.lang.String);
-  public android.support.v17.leanback.widget.HeaderItem(java.lang.String, java.lang.String);
+  public android.support.v17.leanback.widget.HeaderItem(long, java.lang.String);
+  public android.support.v17.leanback.widget.HeaderItem(java.lang.String);
@@ -6 +5,0 @@
-  public final java.lang.String getImageUri();
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.HorizontalGridView leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.HorizontalGridView
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.HorizontalGridView	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.HorizontalGridView	2015-08-20 23:41:43.000000000 +0900
@@ -19,0 +20 @@
+  public void onRtlPropertiesChanged(int);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.ListRowPresenter leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.ListRowPresenter
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.ListRowPresenter	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.ListRowPresenter	2015-08-20 23:41:43.000000000 +0900
@@ -8,0 +9,2 @@
+  public void setRecycledPoolSize(android.support.v17.leanback.widget.Presenter, int);
+  public int getRecycledPoolSize(android.support.v17.leanback.widget.Presenter);
@@ -20,0 +23 @@
+  public void setEntranceTransitionState(android.support.v17.leanback.widget.RowPresenter$ViewHolder, boolean);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.PlaybackControlsRow$FastForwardAction leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.PlaybackControlsRow$FastForwardAction
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.PlaybackControlsRow$FastForwardAction	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.PlaybackControlsRow$FastForwardAction	2015-08-20 23:41:43.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.widget.PlaybackControlsRow$FastForwardAction extends android.support.v17.leanback.widget.Action {
+public class android.support.v17.leanback.widget.PlaybackControlsRow$FastForwardAction extends android.support.v17.leanback.widget.PlaybackControlsRow$MultiAction {
@@ -2,0 +3 @@
+  public android.support.v17.leanback.widget.PlaybackControlsRow$FastForwardAction(android.content.Context, int);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.PlaybackControlsRow$MultiAction leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.PlaybackControlsRow$MultiAction
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.PlaybackControlsRow$MultiAction	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.PlaybackControlsRow$MultiAction	2015-08-20 23:41:43.000000000 +0900
@@ -5 +5,2 @@
-  public int getNumberOfDrawables();
+  public void setSecondaryLabels(java.lang.String[]);
+  public int getActionCount();
@@ -7,0 +9 @@
+  public java.lang.String getSecondaryLabel(int);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.PlaybackControlsRow$RewindAction leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.PlaybackControlsRow$RewindAction
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.PlaybackControlsRow$RewindAction	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.PlaybackControlsRow$RewindAction	2015-08-20 23:41:43.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.widget.PlaybackControlsRow$RewindAction extends android.support.v17.leanback.widget.Action {
+public class android.support.v17.leanback.widget.PlaybackControlsRow$RewindAction extends android.support.v17.leanback.widget.PlaybackControlsRow$MultiAction {
@@ -2,0 +3 @@
+  public android.support.v17.leanback.widget.PlaybackControlsRow$RewindAction(android.content.Context, int);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.RowPresenter leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.RowPresenter
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.RowPresenter	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.RowPresenter	2015-08-20 23:41:43.000000000 +0900
@@ -27,0 +28 @@
+  public void setEntranceTransitionState(android.support.v17.leanback.widget.RowPresenter$ViewHolder, boolean);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.SparseArrayObjectAdapter leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.SparseArrayObjectAdapter
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.SparseArrayObjectAdapter	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.SparseArrayObjectAdapter	2015-08-20 23:41:43.000000000 +0900
@@ -0,0 +1,14 @@
+public class android.support.v17.leanback.widget.SparseArrayObjectAdapter extends android.support.v17.leanback.widget.ObjectAdapter {
+  public android.support.v17.leanback.widget.SparseArrayObjectAdapter(android.support.v17.leanback.widget.PresenterSelector);
+  public android.support.v17.leanback.widget.SparseArrayObjectAdapter(android.support.v17.leanback.widget.Presenter);
+  public android.support.v17.leanback.widget.SparseArrayObjectAdapter();
+  public int size();
+  public java.lang.Object get(int);
+  public int indexOf(java.lang.Object);
+  public int indexOf(int);
+  public void notifyArrayItemRangeChanged(int, int);
+  public void set(int, java.lang.Object);
+  public void clear(int);
+  public void clear();
+  public java.lang.Object lookup(int);
+}
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.TitleView leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.TitleView
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.TitleView	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.TitleView	2015-08-20 23:41:43.000000000 +0900
@@ -12,0 +13 @@
+  public void enableAnimation(boolean);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.VerticalGridView leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.VerticalGridView
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.VerticalGridView	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.VerticalGridView	2015-08-20 23:41:43.000000000 +0900
@@ -6,0 +7 @@
+  public void onRtlPropertiesChanged(int);
diff -U 0 -N leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.WindowAlignment$Axis leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.WindowAlignment$Axis
--- leanback-v17-21.0.3_8d35e96a/android.support.v17.leanback.widget.WindowAlignment$Axis	2015-08-20 23:41:43.000000000 +0900
+++ leanback-v17-22.0.0_5260278a/android.support.v17.leanback.widget.WindowAlignment$Axis	2015-08-20 23:41:43.000000000 +0900
@@ -30,0 +31 @@
+  public final void setReversedFlow(boolean);
