diff -U 0 -N leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.FullWidthDetailsOverviewRowPresenter leanback-v17-23.0.0_78516afe/android.support.v17.leanback.widget.FullWidthDetailsOverviewRowPresenter
--- leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.FullWidthDetailsOverviewRowPresenter	2015-09-05 18:52:11.000000000 +0900
+++ leanback-v17-23.0.0_78516afe/android.support.v17.leanback.widget.FullWidthDetailsOverviewRowPresenter	2015-09-05 18:52:11.000000000 +0900
@@ -4,0 +5,2 @@
+  public static final int ALIGN_MODE_START;
+  public static final int ALIGN_MODE_MIDDLE;
@@ -9,2 +11,4 @@
-  public void setBackgroundColor(int);
-  public int getBackgroundColor();
+  public final void setBackgroundColor(int);
+  public final int getBackgroundColor();
+  public final void setActionsBackgroundColor(int);
+  public final int getActionsBackgroundColor();
@@ -14,0 +19,2 @@
+  public final void setAlignmentMode(int);
+  public final int getAlignmentMode();
diff -U 0 -N leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.HorizontalGridView leanback-v17-23.0.0_78516afe/android.support.v17.leanback.widget.HorizontalGridView
--- leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.HorizontalGridView	2015-09-05 18:52:11.000000000 +0900
+++ leanback-v17-23.0.0_78516afe/android.support.v17.leanback.widget.HorizontalGridView	2015-09-05 18:52:11.000000000 +0900
@@ -41,0 +42 @@
+  public android.view.View focusSearch(int);
diff -U 0 -N leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.VerticalGridView leanback-v17-23.0.0_78516afe/android.support.v17.leanback.widget.VerticalGridView
--- leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.VerticalGridView	2015-09-05 18:52:11.000000000 +0900
+++ leanback-v17-23.0.0_78516afe/android.support.v17.leanback.widget.VerticalGridView	2015-09-05 18:52:11.000000000 +0900
@@ -28,0 +29 @@
+  public android.view.View focusSearch(int);
