diff -U 0 -N leanback-v17-22.2.0_4bca7396/android.support.v17.leanback.widget.HorizontalGridView leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.HorizontalGridView
--- leanback-v17-22.2.0_4bca7396/android.support.v17.leanback.widget.HorizontalGridView	2015-09-05 18:52:11.000000000 +0900
+++ leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.HorizontalGridView	2015-09-05 18:52:11.000000000 +0900
@@ -19,0 +20,2 @@
+  public int getExtraLayoutSpace();
+  public void setExtraLayoutSpace(int);
diff -U 0 -N leanback-v17-22.2.0_4bca7396/android.support.v17.leanback.widget.VerticalGridView leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.VerticalGridView
--- leanback-v17-22.2.0_4bca7396/android.support.v17.leanback.widget.VerticalGridView	2015-09-05 18:52:11.000000000 +0900
+++ leanback-v17-22.2.1_4530a28e/android.support.v17.leanback.widget.VerticalGridView	2015-09-05 18:52:11.000000000 +0900
@@ -6,0 +7,2 @@
+  public int getExtraLayoutSpace();
+  public void setExtraLayoutSpace(int);
