diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.Adaptable leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.Adaptable
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.Adaptable	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.Adaptable	1970-01-01 09:00:00.000000000 +0900
@@ -1,3 +0,0 @@
-public interface android.support.v17.leanback.app.Adaptable {
-  public abstract java.lang.Object getAdapter(java.lang.Class);
-}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrandedFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrandedFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrandedFragment	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrandedFragment	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,26 @@
+public class android.support.v17.leanback.app.BrandedFragment extends android.app.Fragment {
+  public android.support.v17.leanback.app.BrandedFragment();
+  public android.view.View onInflateTitleView(android.view.LayoutInflater, android.view.ViewGroup, android.os.Bundle);
+  public void installTitleView(android.view.LayoutInflater, android.view.ViewGroup, android.os.Bundle);
+  public void setTitleView(android.view.View);
+  public android.view.View getTitleView();
+  public android.support.v17.leanback.widget.TitleViewAdapter getTitleViewAdapter();
+  public void onSaveInstanceState(android.os.Bundle);
+  public void onViewCreated(android.view.View, android.os.Bundle);
+  public void onDestroyView();
+  public void showTitle(boolean);
+  public void showTitle(int);
+  public void setBadgeDrawable(android.graphics.drawable.Drawable);
+  public android.graphics.drawable.Drawable getBadgeDrawable();
+  public void setTitle(java.lang.CharSequence);
+  public java.lang.CharSequence getTitle();
+  public void setOnSearchClickedListener(android.view.View$OnClickListener);
+  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
+  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
+  public void setSearchAffordanceColor(int);
+  public int getSearchAffordanceColor();
+  public void onStart();
+  public void onPause();
+  public void onResume();
+  public final boolean isShowingTitle();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrandedSupportFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrandedSupportFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrandedSupportFragment	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrandedSupportFragment	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,26 @@
+public class android.support.v17.leanback.app.BrandedSupportFragment extends android.support.v4.app.Fragment {
+  public android.support.v17.leanback.app.BrandedSupportFragment();
+  public android.view.View onInflateTitleView(android.view.LayoutInflater, android.view.ViewGroup, android.os.Bundle);
+  public void installTitleView(android.view.LayoutInflater, android.view.ViewGroup, android.os.Bundle);
+  public void setTitleView(android.view.View);
+  public android.view.View getTitleView();
+  public android.support.v17.leanback.widget.TitleViewAdapter getTitleViewAdapter();
+  public void onSaveInstanceState(android.os.Bundle);
+  public void onViewCreated(android.view.View, android.os.Bundle);
+  public void onDestroyView();
+  public void showTitle(boolean);
+  public void showTitle(int);
+  public void setBadgeDrawable(android.graphics.drawable.Drawable);
+  public android.graphics.drawable.Drawable getBadgeDrawable();
+  public void setTitle(java.lang.CharSequence);
+  public java.lang.CharSequence getTitle();
+  public void setOnSearchClickedListener(android.view.View$OnClickListener);
+  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
+  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
+  public void setSearchAffordanceColor(int);
+  public int getSearchAffordanceColor();
+  public void onStart();
+  public void onPause();
+  public void onResume();
+  public final boolean isShowingTitle();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment	2016-05-16 18:10:52.000000000 +0900
@@ -42,11 +41,0 @@
-  public void onResume();
-  public void onPause();
-  public int getSearchAffordanceColor();
-  public void setSearchAffordanceColor(int);
-  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
-  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
-  public void setOnSearchClickedListener(android.view.View$OnClickListener);
-  public java.lang.String getTitle();
-  public void setTitle(java.lang.String);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$FragmentHost leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$FragmentHost
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$FragmentHost	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$FragmentHost	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,5 @@
+public interface android.support.v17.leanback.app.BrowseFragment$FragmentHost {
+  public abstract void notifyViewCreated(android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter);
+  public abstract void notifyDataReady(android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter);
+  public abstract void showTitleView(boolean);
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter<T leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter<T
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter<T	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter<T	2016-05-16 18:10:52.000000000 +0900
@@ -12,0 +13 @@
+  public final android.support.v17.leanback.app.BrowseFragment$FragmentHost getFragmentHost();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapterProvider leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapterProvider
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapterProvider	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapterProvider	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapterProvider {
+  public abstract android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter getMainFragmentAdapter();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapterProvider leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapterProvider
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapterProvider	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapterProvider	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapterProvider {
+  public abstract android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapter getMainFragmentRowsAdapter();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment	2016-05-16 18:10:52.000000000 +0900
@@ -42,11 +41,0 @@
-  public void onResume();
-  public void onPause();
-  public int getSearchAffordanceColor();
-  public void setSearchAffordanceColor(int);
-  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
-  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
-  public void setOnSearchClickedListener(android.view.View$OnClickListener);
-  public java.lang.String getTitle();
-  public void setTitle(java.lang.String);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$FragmentHost leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$FragmentHost
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$FragmentHost	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$FragmentHost	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,5 @@
+public interface android.support.v17.leanback.app.BrowseSupportFragment$FragmentHost {
+  public abstract void notifyViewCreated(android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter);
+  public abstract void notifyDataReady(android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter);
+  public abstract void showTitleView(boolean);
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter<T leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter<T
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter<T	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter<T	2016-05-16 18:10:52.000000000 +0900
@@ -12,0 +13 @@
+  public final android.support.v17.leanback.app.BrowseSupportFragment$FragmentHost getFragmentHost();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapterProvider leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapterProvider
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapterProvider	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapterProvider	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapterProvider {
+  public abstract android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter getMainFragmentAdapter();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapterProvider leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapterProvider
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapterProvider	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapterProvider	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapterProvider {
+  public abstract android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapter getMainFragmentRowsAdapter();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.DetailsFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.DetailsFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.DetailsFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.DetailsFragment	2016-05-16 18:10:52.000000000 +0900
@@ -9,0 +10 @@
+  public android.view.View onInflateTitleView(android.view.LayoutInflater, android.view.ViewGroup, android.os.Bundle);
@@ -17,13 +17,0 @@
-  public void onResume();
-  public void onPause();
-  public int getSearchAffordanceColor();
-  public void setSearchAffordanceColor(int);
-  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
-  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
-  public void setOnSearchClickedListener(android.view.View$OnClickListener);
-  public java.lang.String getTitle();
-  public void setTitle(java.lang.String);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
-  public void onDestroyView();
-  public void onSaveInstanceState(android.os.Bundle);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.DetailsSupportFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.DetailsSupportFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.DetailsSupportFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.DetailsSupportFragment	2016-05-16 18:10:52.000000000 +0900
@@ -9,0 +10 @@
+  public android.view.View onInflateTitleView(android.view.LayoutInflater, android.view.ViewGroup, android.os.Bundle);
@@ -17,13 +17,0 @@
-  public void onResume();
-  public void onPause();
-  public int getSearchAffordanceColor();
-  public void setSearchAffordanceColor(int);
-  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
-  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
-  public void setOnSearchClickedListener(android.view.View$OnClickListener);
-  public java.lang.String getTitle();
-  public void setTitle(java.lang.String);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
-  public void onDestroyView();
-  public void onSaveInstanceState(android.os.Bundle);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.ErrorFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.ErrorFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.ErrorFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.ErrorFragment	2016-05-16 18:10:52.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.ErrorFragment extends android.app.Fragment {
+public class android.support.v17.leanback.app.ErrorFragment extends android.support.v17.leanback.app.BrandedFragment {
@@ -3,4 +2,0 @@
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setTitle(java.lang.String);
-  public java.lang.String getTitle();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.ErrorSupportFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.ErrorSupportFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.ErrorSupportFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.ErrorSupportFragment	2016-05-16 18:10:52.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.ErrorSupportFragment extends android.support.v4.app.Fragment {
+public class android.support.v17.leanback.app.ErrorSupportFragment extends android.support.v17.leanback.app.BrandedSupportFragment {
@@ -3,4 +2,0 @@
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setTitle(java.lang.String);
-  public java.lang.String getTitle();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.GuidedStepFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.GuidedStepFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.GuidedStepFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.GuidedStepFragment	2016-05-16 18:10:52.000000000 +0900
@@ -23,0 +24 @@
+  public void onGuidedActionEditCanceled(android.support.v17.leanback.widget.GuidedAction);
@@ -53,0 +55 @@
+  public void onResume();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.GuidedStepSupportFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.GuidedStepSupportFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.GuidedStepSupportFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.GuidedStepSupportFragment	2016-05-16 18:10:52.000000000 +0900
@@ -23,0 +24 @@
+  public void onGuidedActionEditCanceled(android.support.v17.leanback.widget.GuidedAction);
@@ -53,0 +55 @@
+  public void onResume();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.RowsFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.RowsFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.RowsFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.RowsFragment	2016-05-16 18:10:52.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.RowsFragment extends android.support.v17.leanback.app.BaseRowFragment implements android.support.v17.leanback.app.Adaptable {
+public class android.support.v17.leanback.app.RowsFragment extends android.support.v17.leanback.app.BaseRowFragment implements android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapterProvider,android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapterProvider {
@@ -3 +3,2 @@
-  public java.lang.Object getAdapter(java.lang.Class);
+  public android.support.v17.leanback.app.BrowseFragment$MainFragmentAdapter getMainFragmentAdapter();
+  public android.support.v17.leanback.app.BrowseFragment$MainFragmentRowsAdapter getMainFragmentRowsAdapter();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.RowsSupportFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.RowsSupportFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.RowsSupportFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.RowsSupportFragment	2016-05-16 18:10:52.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.app.RowsSupportFragment extends android.support.v17.leanback.app.BaseRowSupportFragment implements android.support.v17.leanback.app.Adaptable {
+public class android.support.v17.leanback.app.RowsSupportFragment extends android.support.v17.leanback.app.BaseRowSupportFragment implements android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapterProvider,android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapterProvider {
@@ -3 +3,2 @@
-  public java.lang.Object getAdapter(java.lang.Class);
+  public android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentAdapter getMainFragmentAdapter();
+  public android.support.v17.leanback.app.BrowseSupportFragment$MainFragmentRowsAdapter getMainFragmentRowsAdapter();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.VerticalGridFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.VerticalGridFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.VerticalGridFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.VerticalGridFragment	2016-05-16 18:10:52.000000000 +0900
@@ -17,12 +16,0 @@
-  public void onResume();
-  public void onPause();
-  public int getSearchAffordanceColor();
-  public void setSearchAffordanceColor(int);
-  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
-  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
-  public void setOnSearchClickedListener(android.view.View$OnClickListener);
-  public java.lang.String getTitle();
-  public void setTitle(java.lang.String);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
-  public void onSaveInstanceState(android.os.Bundle);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.VerticalGridSupportFragment leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.VerticalGridSupportFragment
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.app.VerticalGridSupportFragment	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.app.VerticalGridSupportFragment	2016-05-16 18:10:52.000000000 +0900
@@ -17,12 +16,0 @@
-  public void onResume();
-  public void onPause();
-  public int getSearchAffordanceColor();
-  public void setSearchAffordanceColor(int);
-  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
-  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
-  public void setOnSearchClickedListener(android.view.View$OnClickListener);
-  public java.lang.String getTitle();
-  public void setTitle(java.lang.String);
-  public android.graphics.drawable.Drawable getBadgeDrawable();
-  public void setBadgeDrawable(android.graphics.drawable.Drawable);
-  public void onSaveInstanceState(android.os.Bundle);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaItemPresenter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaItemPresenter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaItemPresenter	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaItemPresenter	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,6 @@
+public abstract class android.support.v17.leanback.widget.AbstractMediaItemPresenter extends android.support.v17.leanback.widget.RowPresenter {
+  public android.support.v17.leanback.widget.AbstractMediaItemPresenter(android.content.Context, int);
+  public android.support.v17.leanback.widget.AbstractMediaItemPresenter();
+  public boolean isUsingDefaultSelectEffect();
+  public void setBackgroundColor(int);
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaItemPresenter$ViewHolder leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaItemPresenter$ViewHolder
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaItemPresenter$ViewHolder	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaItemPresenter$ViewHolder	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v17.leanback.widget.AbstractMediaItemPresenter$ViewHolder extends android.support.v17.leanback.widget.RowPresenter$ViewHolder {
+  public android.support.v17.leanback.widget.AbstractMediaItemPresenter$ViewHolder(android.view.View);
+  public android.widget.TextView getTrackNumberView();
+  public android.widget.TextView getTrackNameView();
+  public android.widget.TextView getTrackDurationView();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,6 @@
+public abstract class android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter extends android.support.v17.leanback.widget.RowPresenter {
+  public android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter(android.content.Context, int);
+  public android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter();
+  public boolean isUsingDefaultSelectEffect();
+  public void setBackgroundColor(int);
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter$ViewHolder leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter$ViewHolder
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter$ViewHolder	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter$ViewHolder	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter$ViewHolder extends android.support.v17.leanback.widget.RowPresenter$ViewHolder {
+  public android.support.v17.leanback.widget.AbstractMediaListHeaderPresenter$ViewHolder(android.view.View);
+  public android.widget.TextView getHeaderView();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.Action leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.Action
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.Action	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.Action	2016-05-16 18:10:52.000000000 +0900
@@ -1,0 +2 @@
+  public static final long NO_ID;
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.DividerPresenter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.DividerPresenter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.DividerPresenter	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.DividerPresenter	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,7 @@
+public class android.support.v17.leanback.widget.DividerPresenter extends android.support.v17.leanback.widget.Presenter {
+  public android.support.v17.leanback.widget.DividerPresenter();
+  public android.support.v17.leanback.widget.DividerPresenter(int);
+  public android.support.v17.leanback.widget.Presenter$ViewHolder onCreateViewHolder(android.view.ViewGroup);
+  public void onBindViewHolder(android.support.v17.leanback.widget.Presenter$ViewHolder, java.lang.Object);
+  public void onUnbindViewHolder(android.support.v17.leanback.widget.Presenter$ViewHolder);
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.DividerRow leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.DividerRow
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.DividerRow	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.DividerRow	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,4 @@
+public class android.support.v17.leanback.widget.DividerRow extends android.support.v17.leanback.widget.Row {
+  public android.support.v17.leanback.widget.DividerRow();
+  public final boolean isRenderedAsRowView();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedAction leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedAction
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedAction	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedAction	2016-05-16 18:10:52.000000000 +0900
@@ -44,0 +45,3 @@
+  public final boolean isAutoSaveRestoreEnabled();
+  public void onSaveInstanceState(android.os.Bundle, java.lang.String);
+  public void onRestoreInstanceState(android.os.Bundle, java.lang.String);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedAction$BuilderBase<B leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedAction$BuilderBase<B
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedAction$BuilderBase<B	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedAction$BuilderBase<B	2016-05-16 18:10:52.000000000 +0900
@@ -32,0 +33 @@
+  public B autoSaveRestoreEnabled(boolean);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedActionAdapter$EditListener leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedActionAdapter$EditListener
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedActionAdapter$EditListener	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedActionAdapter$EditListener	2016-05-16 18:10:52.000000000 +0900
@@ -2 +2,2 @@
-  public abstract long onGuidedActionEdited(android.support.v17.leanback.widget.GuidedAction);
+  public abstract void onGuidedActionEditCanceled(android.support.v17.leanback.widget.GuidedAction);
+  public abstract long onGuidedActionEditedAndProceed(android.support.v17.leanback.widget.GuidedAction);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedActionsStylist$ViewHolder leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedActionsStylist$ViewHolder
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedActionsStylist$ViewHolder	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedActionsStylist$ViewHolder	2016-05-16 18:10:52.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.widget.GuidedActionsStylist$ViewHolder extends android.support.v7.widget.RecyclerView$ViewHolder {
+public class android.support.v17.leanback.widget.GuidedActionsStylist$ViewHolder extends android.support.v7.widget.RecyclerView$ViewHolder implements android.support.v17.leanback.widget.FacetProvider {
@@ -19,0 +20 @@
+  public java.lang.Object getFacet(java.lang.Class<?>);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedDatePickerAction leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedDatePickerAction
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.GuidedDatePickerAction	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.GuidedDatePickerAction	2016-05-16 18:10:52.000000000 +0900
@@ -7,0 +8,2 @@
+  public void onSaveInstanceState(android.os.Bundle, java.lang.String);
+  public void onRestoreInstanceState(android.os.Bundle, java.lang.String);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.HeaderItem leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.HeaderItem
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.HeaderItem	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.HeaderItem	2016-05-16 18:10:52.000000000 +0900
@@ -5,0 +6,2 @@
+  public java.lang.CharSequence getContentDescription();
+  public void setContentDescription(java.lang.CharSequence);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.InvisibleRowPresenter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.InvisibleRowPresenter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.InvisibleRowPresenter	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.InvisibleRowPresenter	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,3 @@
+public class android.support.v17.leanback.widget.InvisibleRowPresenter extends android.support.v17.leanback.widget.RowPresenter {
+  public android.support.v17.leanback.widget.InvisibleRowPresenter();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.ItemAlignmentFacet$ItemAlignmentDef leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.ItemAlignmentFacet$ItemAlignmentDef
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.ItemAlignmentFacet$ItemAlignmentDef	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.ItemAlignmentFacet$ItemAlignmentDef	2016-05-16 18:10:52.000000000 +0900
@@ -12,0 +13,2 @@
+  public final void setAlignedToTextViewBaseline(boolean);
+  public boolean isAlignedToTextViewBaseLine();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.ListRow leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.ListRow
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.ListRow	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.ListRow	2016-05-16 18:10:52.000000000 +0900
@@ -5,0 +6,2 @@
+  public java.lang.CharSequence getContentDescription();
+  public void setContentDescription(java.lang.CharSequence);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.ListRowPresenter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.ListRowPresenter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.ListRowPresenter	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.ListRowPresenter	2016-05-16 18:10:52.000000000 +0900
@@ -11,0 +12 @@
+  public void setNumRows(int);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.PageRow leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.PageRow
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.PageRow	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.PageRow	2016-05-16 18:10:52.000000000 +0900
@@ -2,0 +3 @@
+  public final boolean isRenderedAsRowView();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.PageRowPresenter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.PageRowPresenter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.PageRowPresenter	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.PageRowPresenter	1970-01-01 09:00:00.000000000 +0900
@@ -1,3 +0,0 @@
-public class android.support.v17.leanback.widget.PageRowPresenter extends android.support.v17.leanback.widget.RowPresenter {
-  public android.support.v17.leanback.widget.PageRowPresenter();
-}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.Row leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.Row
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.Row	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.Row	2016-05-16 18:10:52.000000000 +0900
@@ -8,0 +9 @@
+  public boolean isRenderedAsRowView();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.RowHeaderPresenter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.RowHeaderPresenter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.RowHeaderPresenter	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.RowHeaderPresenter	2016-05-16 18:10:52.000000000 +0900
@@ -3,0 +4 @@
+  public android.support.v17.leanback.widget.RowHeaderPresenter(int, boolean);
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.SectionRow leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.SectionRow
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.SectionRow	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.SectionRow	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,6 @@
+public class android.support.v17.leanback.widget.SectionRow extends android.support.v17.leanback.widget.Row {
+  public android.support.v17.leanback.widget.SectionRow(android.support.v17.leanback.widget.HeaderItem);
+  public android.support.v17.leanback.widget.SectionRow(long, java.lang.String);
+  public android.support.v17.leanback.widget.SectionRow(java.lang.String);
+  public final boolean isRenderedAsRowView();
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleHelper leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleHelper
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleHelper	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleHelper	2016-05-16 18:10:52.000000000 +0900
@@ -2 +2 @@
-  public android.support.v17.leanback.widget.TitleHelper(android.view.ViewGroup, android.support.v17.leanback.widget.TitleView);
+  public android.support.v17.leanback.widget.TitleHelper(android.view.ViewGroup, android.view.View);
@@ -5 +5 @@
-  public android.support.v17.leanback.widget.TitleView getTitleView();
+  public android.view.View getTitleView();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleView leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleView
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleView	2016-05-16 18:10:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleView	2016-05-16 18:10:52.000000000 +0900
@@ -1 +1 @@
-public class android.support.v17.leanback.widget.TitleView extends android.widget.FrameLayout {
+public class android.support.v17.leanback.widget.TitleView extends android.widget.FrameLayout implements android.support.v17.leanback.widget.TitleViewAdapter$Provider {
@@ -5 +5 @@
-  public void setTitle(java.lang.String);
+  public void setTitle(java.lang.CharSequence);
@@ -13,0 +14,2 @@
+  public void updateComponentsVisibility(int);
+  public android.support.v17.leanback.widget.TitleViewAdapter getTitleViewAdapter();
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleViewAdapter leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleViewAdapter
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleViewAdapter	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleViewAdapter	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,16 @@
+public abstract class android.support.v17.leanback.widget.TitleViewAdapter {
+  public static final int BRANDING_VIEW_VISIBLE;
+  public static final int SEARCH_VIEW_VISIBLE;
+  public static final int FULL_VIEW_VISIBLE;
+  public android.support.v17.leanback.widget.TitleViewAdapter();
+  public void setTitle(java.lang.CharSequence);
+  public java.lang.CharSequence getTitle();
+  public void setBadgeDrawable(android.graphics.drawable.Drawable);
+  public android.graphics.drawable.Drawable getBadgeDrawable();
+  public abstract android.view.View getSearchAffordanceView();
+  public void setOnSearchClickedListener(android.view.View$OnClickListener);
+  public void setSearchAffordanceColors(android.support.v17.leanback.widget.SearchOrbView$Colors);
+  public android.support.v17.leanback.widget.SearchOrbView$Colors getSearchAffordanceColors();
+  public void setAnimationEnabled(boolean);
+  public void updateComponentsVisibility(int);
+}
diff -U 0 -N leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleViewAdapter$Provider leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleViewAdapter$Provider
--- leanback-v17-24.0.0-alpha2_7197b9e6/android.support.v17.leanback.widget.TitleViewAdapter$Provider	1970-01-01 09:00:00.000000000 +0900
+++ leanback-v17-24.0.0-beta1_eced1552/android.support.v17.leanback.widget.TitleViewAdapter$Provider	2016-05-16 18:10:52.000000000 +0900
@@ -0,0 +1,3 @@
+public interface android.support.v17.leanback.widget.TitleViewAdapter$Provider {
+  public abstract android.support.v17.leanback.widget.TitleViewAdapter getTitleViewAdapter();
+}
