diff -U 0 -N design-24.0.0-alpha2_502c4742/android.support.design.widget.AppBarLayout$Behavior$SavedState design-24.0.0-beta1_7f9287fe/android.support.design.widget.AppBarLayout$Behavior$SavedState
--- design-24.0.0-alpha2_502c4742/android.support.design.widget.AppBarLayout$Behavior$SavedState	2016-05-16 18:10:00.000000000 +0900
+++ design-24.0.0-beta1_7f9287fe/android.support.design.widget.AppBarLayout$Behavior$SavedState	2016-05-16 18:10:46.000000000 +0900
@@ -1 +1 @@
-public class android.support.design.widget.AppBarLayout$Behavior$SavedState extends android.view.View$BaseSavedState {
+public class android.support.design.widget.AppBarLayout$Behavior$SavedState extends android.support.v4.view.AbsSavedState {
diff -U 0 -N design-24.0.0-alpha2_502c4742/android.support.design.widget.BottomSheetBehavior$SavedState design-24.0.0-beta1_7f9287fe/android.support.design.widget.BottomSheetBehavior$SavedState
--- design-24.0.0-alpha2_502c4742/android.support.design.widget.BottomSheetBehavior$SavedState	2016-05-16 18:10:00.000000000 +0900
+++ design-24.0.0-beta1_7f9287fe/android.support.design.widget.BottomSheetBehavior$SavedState	2016-05-16 18:10:46.000000000 +0900
@@ -1 +1 @@
-public class android.support.design.widget.BottomSheetBehavior$SavedState extends android.view.View$BaseSavedState {
+public class android.support.design.widget.BottomSheetBehavior$SavedState extends android.support.v4.view.AbsSavedState {
@@ -3,0 +4 @@
+  public android.support.design.widget.BottomSheetBehavior$SavedState(android.os.Parcel, java.lang.ClassLoader);
diff -U 0 -N design-24.0.0-alpha2_502c4742/android.support.design.widget.CollapsingToolbarLayout design-24.0.0-beta1_7f9287fe/android.support.design.widget.CollapsingToolbarLayout
--- design-24.0.0-alpha2_502c4742/android.support.design.widget.CollapsingToolbarLayout	2016-05-16 18:10:00.000000000 +0900
+++ design-24.0.0-beta1_7f9287fe/android.support.design.widget.CollapsingToolbarLayout	2016-05-16 18:10:46.000000000 +0900
@@ -41,0 +42,4 @@
+  public void setScrimVisibleHeightTrigger(int);
+  public int getScrimVisibleHeightTrigger();
+  public void setScrimAnimationDuration(int);
+  public int getScrimAnimationDuration();
diff -U 0 -N design-24.0.0-alpha2_502c4742/android.support.design.widget.CoordinatorLayout$SavedState design-24.0.0-beta1_7f9287fe/android.support.design.widget.CoordinatorLayout$SavedState
--- design-24.0.0-alpha2_502c4742/android.support.design.widget.CoordinatorLayout$SavedState	2016-05-16 18:10:00.000000000 +0900
+++ design-24.0.0-beta1_7f9287fe/android.support.design.widget.CoordinatorLayout$SavedState	2016-05-16 18:10:46.000000000 +0900
@@ -1 +1 @@
-public class android.support.design.widget.CoordinatorLayout$SavedState extends android.view.View$BaseSavedState {
+public class android.support.design.widget.CoordinatorLayout$SavedState extends android.support.v4.view.AbsSavedState {
diff -U 0 -N design-24.0.0-alpha2_502c4742/android.support.design.widget.FloatingActionButton$Behavior design-24.0.0-beta1_7f9287fe/android.support.design.widget.FloatingActionButton$Behavior
--- design-24.0.0-alpha2_502c4742/android.support.design.widget.FloatingActionButton$Behavior	2016-05-16 18:10:00.000000000 +0900
+++ design-24.0.0-beta1_7f9287fe/android.support.design.widget.FloatingActionButton$Behavior	2016-05-16 18:10:46.000000000 +0900
@@ -2,0 +3 @@
+  public android.support.design.widget.FloatingActionButton$Behavior(android.content.Context, android.util.AttributeSet);
@@ -4,0 +6 @@
+  public void onDependentViewRemoved(android.support.design.widget.CoordinatorLayout, android.support.design.widget.FloatingActionButton, android.view.View);
@@ -6,0 +9 @@
+  public void onDependentViewRemoved(android.support.design.widget.CoordinatorLayout, android.view.View, android.view.View);
diff -U 0 -N design-24.0.0-alpha2_502c4742/android.support.design.widget.NavigationView$SavedState design-24.0.0-beta1_7f9287fe/android.support.design.widget.NavigationView$SavedState
--- design-24.0.0-alpha2_502c4742/android.support.design.widget.NavigationView$SavedState	2016-05-16 18:10:00.000000000 +0900
+++ design-24.0.0-beta1_7f9287fe/android.support.design.widget.NavigationView$SavedState	2016-05-16 18:10:46.000000000 +0900
@@ -1 +1 @@
-public class android.support.design.widget.NavigationView$SavedState extends android.view.View$BaseSavedState {
+public class android.support.design.widget.NavigationView$SavedState extends android.support.v4.view.AbsSavedState {
