diff -U 0 -N design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.AppBarLayout$Behavior design-24.0.0-alpha2_502c4742/android.support.design.widget.AppBarLayout$Behavior
--- design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.AppBarLayout$Behavior	2016-04-16 19:37:19.000000000 +0900
+++ design-24.0.0-alpha2_502c4742/android.support.design.widget.AppBarLayout$Behavior	2016-04-16 19:37:19.000000000 +0900
@@ -9,0 +10 @@
+  public boolean onMeasureChild(android.support.design.widget.CoordinatorLayout, android.support.design.widget.AppBarLayout, int, int, int, int);
@@ -24,0 +26 @@
+  public boolean onMeasureChild(android.support.design.widget.CoordinatorLayout, android.view.View, int, int, int, int);
diff -U 0 -N design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.BottomSheetDialogFragment design-24.0.0-alpha2_502c4742/android.support.design.widget.BottomSheetDialogFragment
--- design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.BottomSheetDialogFragment	2016-04-16 19:37:19.000000000 +0900
+++ design-24.0.0-alpha2_502c4742/android.support.design.widget.BottomSheetDialogFragment	2016-04-16 19:37:19.000000000 +0900
@@ -4 +3,0 @@
-  public void setupDialog(android.app.Dialog, int);
diff -U 0 -N design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.FloatingActionButton design-24.0.0-alpha2_502c4742/android.support.design.widget.FloatingActionButton
--- design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.FloatingActionButton	2016-04-16 19:37:19.000000000 +0900
+++ design-24.0.0-alpha2_502c4742/android.support.design.widget.FloatingActionButton	2016-04-16 19:37:19.000000000 +0900
@@ -1,0 +2,3 @@
+  public static final int SIZE_MINI;
+  public static final int SIZE_NORMAL;
+  public static final int SIZE_AUTO;
@@ -12,0 +16 @@
+  public void setImageResource(int);
@@ -18,0 +23,2 @@
+  public void setSize(int);
+  public int getSize();
diff -U 0 -N design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.FloatingActionButton$Size design-24.0.0-alpha2_502c4742/android.support.design.widget.FloatingActionButton$Size
--- design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.FloatingActionButton$Size	1970-01-01 09:00:00.000000000 +0900
+++ design-24.0.0-alpha2_502c4742/android.support.design.widget.FloatingActionButton$Size	2016-04-16 19:37:19.000000000 +0900
@@ -0,0 +1,2 @@
+public interface android.support.design.widget.FloatingActionButton$Size extends java.lang.annotation.Annotation {
+}
diff -U 0 -N design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.TabLayout design-24.0.0-alpha2_502c4742/android.support.design.widget.TabLayout
--- design-24.0.0-alpha1_f37d7c3c/android.support.design.widget.TabLayout	2016-04-16 19:37:19.000000000 +0900
+++ design-24.0.0-alpha2_502c4742/android.support.design.widget.TabLayout	2016-04-16 19:37:19.000000000 +0900
@@ -16,0 +17,2 @@
+  public void addOnTabSelectedListener(android.support.design.widget.TabLayout$OnTabSelectedListener);
+  public void removeOnTabSelectedListener(android.support.design.widget.TabLayout$OnTabSelectedListener);
@@ -31,0 +34 @@
+  public void setupWithViewPager(android.support.v4.view.ViewPager, boolean);
